Struggling Author mission chain (#10680)
endless-sky/endless-sky: data/coalition/coalition missions.txt

				decline

> mission "Heliarchs Buy Jump Drive Mission"

Author: ```suggestion

mission "Heliarchs Buy Jump Drive Mission"
```

	name "Give Kachachee Cultural Information"
	description "A group of Coalition citizens have asked you for information on the human race."
	source "Ki Patek Ka"
	to offer
		has "First Contact: Coalition: done"
		has "human cultural data"
	on offer
		conversation
			`As you walk through the spaceport, you see a group of Coalition citizens seated by a table speaking to each other. One of them glances toward you. As if something suddenly clicked, the group quickly rises from their seats and begin approaching you.`
			choice
				`	"Excuse me, what do you all want?"`
			`	"Greetings, human friend. Collectors of literature, we are. Curious about your race, and the books they have written, we are, so for you, a mission we have," says a Kimek, who then introduces themselves as Kachachee. "Literature of many kinds, we read, from science fiction to tales of alien worlds outside of our homeland.`
			`	"A small mission, it is. Knowledge from your race, we seek. Pay you <payment> and be very grateful, we would, if accept this mission, you will," explains the citizen.`
			choice
				`	"Sorry, I'm not interested."`
					decline
				`	"I'm in. What do you want me to collect?"`

> 			`	"Anything. A translation device we have, so translate the human language into our own, we are able to. As for the topic, a great story about oppressing dictatorships or tyranny, we want. Histories of humanity's resistance movements, we seek."`

Author: ```suggestion
			`	"Anything. A translation device we have, so translate the human language into our own, we are able to. As for the topic, a great story about dictatorships and brave battles, tyrants and rebels, we want. Histories of humanity's resistance movements, we seek, but not anti-Heliarch, must they be."`
```

			action
				event "coalition citizens discover fangvin" 14 27
				payment 180000
			`	"Thank you. Enlighten me and my friends, this datachip will. And as promised, a chip of 180,000 credits, we have." Kachachee places it within your hand.`
				decline

event "coalition citizens discover fangvin"

mission "Coalition: Struggling Author - Book Discovered"
	name "Look for Anna Fangvin"
	description "Get in contact with Anna Fangvin, an author in which the group of Coalition citizens have shown interest in."
	source
		government "Coalition"
	destination "Zug"
	to offer
		has "event: coalition citizens discover fangvin"
	on offer
		conversation
			`Upon landing in the spaceport, you get a message from the same group of Coalition citizens. You recall that you had provided them with a datachip containing the a copy of Alexandria's archive.`
			``
			`		Dear <first>,`
			`	Interested in an author from the datachip, we are. 'Anna Fangvin', her name is. While the full book, access it we could not, the summary, about your race's actions against the oppressors, captivate us it did. To converse with her, we wish, and a copy of the book, we desire.`

> 			`	On Ki Patek Ka we live, so the book, once in your possession, it is, contact us, and meet you at the spaceport, we will.`

Author: ```suggestion
```

			choice
				`	(Look for Anna Fangvin.)`
				`	(Disregard the message.)`
					decline

> 			`	A quick look through your encyclopedia reveals Anna Fangvin to be an author from <destination>. You decide to meet her there for an interview.`

Author: ```suggestion
			`	A quick look through your encyclopedia reveals nothing. However, checking a copy of the data chip that you gave the Coalition citizens reveals Anna Fangvin to be an author from <destination>. You decide to meet her there for an interview.`
```

			choice
				`	(Leave.)`
			`	You return to the spaceport the way you came and return to your ship. You are slightly anxious about the outcome of the process and whether Anna's book will even make it through the process and be published, but you have more important things to attend to for the next six months.`
				decline
			label stay
			`	"I need to stay here so I can answer their questions during the review process. Don't worry about me."`
			choice
				`	"Got it. Good luck publishing your book."`
					goto thanks

event "beyond the hyperdrive is published"

mission "Coalition: Struggling Author 5A"
	landing
	source
		not near "Quaru" 0 100
	destination "Inmost Blue"
	to offer
		has "event: beyond the hyperdrive is published"
		not "Coalition: Struggling Author 5B: offered"
	on offer
		conversation
			`Checking your computer feed for the latest galactic news and trade prices, you get a reminder from your computer.`
			`	You read it, and recall that you had helped an author named Anna Fangvin get her book, 'Beyond the Hyperdrive,' published about half a year ago in Coalition space. You should return to <destination> and check on her.`
				accept
	on complete
		payment 50000

> 		"salary: Beyond the Hyperdrive" = 800

Author: ```suggestion
		"salary: Beyond the Hyperdrive" = 100
```

				accept
	on complete
		set "coalition: struggling author - beyond the hyperdrive"
		conversation
			`Anna gives you the book, and then you leave the ship. The moment you are out, three Kimek approach you and Anna. The Kimek in the middle speaks, and then a brief silence occurs. It is broken as their translation box speaks.`
			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"`
			`	The Coalition is already familiar with you, so you tell them your name while Anna introduces herself. "About what, do you write?" asks the Kimek. "Oh, I write science fiction books," replies Anna.`
			`	The Kimek begin conversing amongst themselves in their native language for a good ten to twenty seconds before they move to face you again.`
			`	Then, switching back to your language, the Kimek explains that Anna is welcome to be with you, but states that she will not be allowed to do much unless she registers. "Register?" asks Anna.`

> 			`	"You have to register yourself with the Coalition before you can do most stuff here. I'll take you to the Ring of Friendship, which is where you can complete the registration process." you say.`

Author: Presumably, you'd want to meet someone you are going to work for before finalizing whether you get the job.

			`	You sit down next to Anna. "These vegetables are so delicious! After months of instant noodles and takeout back home, it finally feels good to eat something that's actually much healthier." she exclaims, closing the computer.`
			choice
				`	"How good is it?"`
					goto good
				`	"What did you usually eat back in human space?"`
			`	"My diet wasn't exactly the best. All I could eat were instant noodles and takeout food; I never could find time in my schedule to cook at home. I was working hard on 'Beyond the Hyperdrive' at the time. This food is better than any instant noodle brand I could ever find on Zug."`
			choice
				`	"What's our plan now?"`
					goto published
				`	"What are we going to do next?"`
					goto published
			label good
			`	"It's very good. Probably better than anything I've had back on Zug."`
			choice
				`	"So, what do you want to do now?"`
			label published
			`	"I want to get my book published here."`
			choice
				`	"Have you found a publishing company yet?"`

> 			`	"No." Anna then begins complaining about how it was extremely difficult for her to communicate with both the restaurant staff and the publishing companies, who did not have a translation box.`

Reviewer: They are common enough that you can talk with most folks you find even if you don't have one yourself, but not everyone has them, rather it's a small % of the population that has the translation devices themselves. I'd say they're at about the point computers were in a few decades ago, in terms of availability (though comparatively a lot more expensive).

Given that, the restaurant staff not having translators of them own would be the expected, whereas the publishing company would definetly have them.

			`	As you listen, you learn that due to the Coalition's censorship and review process, Anna will have to stay for about half a year, during which she will have an apartment to live in.`
			choice
				`	"Do I have to stay with Anna here for half a year?"`
					goto "half a year"
				`	(Say nothing.)`
			label "rest of interview"
			`	As the hours pass, Anna's enthusiastic tone remains strong and constant. The Kimek then takes the book, puts it in a nearby filing cabinet and thanks you and Anna. The three of you exchange pleasantries.`
			`	You leave the interview room, the Heliarchs staying in the room, presumably to continue discussion with the Kimek. As the two of you walk, Anna begins to speak.`
				goto thanks
			label "half a year"
			`	"Required to stay with Anna, you are not, but if you wish to, stay with her you may. To return to human space, you are free to do," says the Kimek.`
				goto "rest of interview"
			label "lounge"

> 			`	You leave the room and walk to the lounge. There are several tables where many Kimek are chatting in their native language, while a few Saryds and Arachs are visibly relaxed. You sit at an empty table and pass the time, occasionally going to the water dispenser to hydrate yourself.`

Reviewer 2: ```suggestion
			`	You leave the room and walk to the lounge. There are several tables where many Kimek are chatting in their native language, while a few Saryds and Arachi are visibly relaxed. You sit at an empty table and pass the time, occasionally going to the water dispenser to hydrate yourself.`
```

				`	"Well, I'm not the only alien."`
					goto "kst-1 not only alien"
				`	"We call ourselves humans, not aliens."`
					goto "kst-1 endonym"
			label "kst-1 glad to meet you"
			`	"Same. 'Akacharra', my name is. To our fine neighborhood, for what purpose have you come here?"`
			choice
				`	"Oh, my friend Anna Fangvin is publishing a book here. I brought her here."`
					goto "kst-1 anna"
			label "kst-1 not only alien"
			`	"If more of you, here there are, then who are they?"`
			choice
				`	"It's just two; my friend Anna Fangvin and I. She's publishing a book in your space. I brought her here."`
					goto "kst-1 anna"
			label "kst-1 endonym"
			`	"'Humans', you are? Know that, I never did. For correcting me and educating me on your fellow kind, I am thankful," says the Kimek. "So, the other human, who is she?"`
			choice
				`	"Oh, that's my friend Anna Fangvin. She's here to publish her book in Coalition space."`
			label "kst-1 anna"
			`	"Anna Fangvin, her name is? An interesting name, it is, and her book, just as interesting it will be when it is released. But know the title of the book, I do not, so if tell me you will, appreciate it I will."`
			choice

> 				`	"It is called 'Beyond the Hyperdrive'. It could very much be years before release, but I'm glad you're taking interest."`

Reviewer 2: ```suggestion
				`	"It is called 'Beyond the Hyperdrive'. It could very well be years before release, but I'm glad you're taking interest."`
```

> 				`	"It is called 'Beyond the Hyperdrive'.`

Reviewer 2: Does this dialog choice really have to be the same as the first one but shorter?

				`	"How did the interview go?"`
			`	"They told me that due to the censorship and review process, I'll have to wait for roughly half a year. Thankfully, they have arranged a small apartment for me to live in. You can go on your own now, but please make sure you come back. If you want to return to human space, please remember to come back and meet me."`
			choice
				`	"Why don't you just go back to human space?"`
					goto stay
				`	"I'll make sure to do that."`
			label thanks
			`	"Thanks again, <first> <last>. I'll forever be grateful to you for helping me. I never thought I'd be successful as an author, but when I met you, you helped me turn my career into something I thought I'd never see."`
			choice
				`	"Just helping out."`
				`	"I'm happy to have helped you."`
			`	"Again, thank you. You will not be needed anymore for the process."`
			choice
				`	(Leave.)`
			`	You return to the spaceport the way you came and return to your ship. You are slightly anxious about the outcome of the process and whether Anna's book will even make it through the process and be published, but you have more important things to attend to for the next six months.`
				decline
			label stay

> 			`	"I need to stay here so I can answer their questions during the review process. Don't worry about me."`

Reviewer 2: She can just do that? Does she not have any sort of connections or obligations in human space? What about her friends and family?

				accept
	on complete
		payment 50000
		"salary: Beyond the Hyperdrive" = 100
		conversation "anna invites you to book signing"

mission "Coalition: Struggling Author 5B"
	landing
	source
		near "Quaru" 0 100
	destination "Inmost Blue"
	to offer
		has "event: beyond the hyperdrive is published"
		not "Coalition: Struggling Author 5A: offered"
	on offer
		conversation
			`As you land on the spaceport, you check your messages. You see a message from Anna Fangvin, the author who you helped about half a year ago. In the message, she explains that her novel, 'Beyond the Hyperdrive,' has been approved. You should return to <destination> and meet her.`
				accept
	on complete
		payment 50000
		"salary: Beyond the Hyperdrive" = 100
		conversation "anna invites you to book signing"

conversation "anna invites you to book signing"
	action
		log "People" "Anna Fangvin" `Anna is a science fiction writer. She was a struggling author in human space, but found success upon publishing her novel 'Beyond the Hyperdrive' in Coalition space. Her book is a bestseller in Kimek space.`

> 		log "Helped somebody named Anna Fangvin get her book published in Coalition space. Apparently, it's a bestseller now. The book was 'Beyond the Hyperdrive', something about humans stealing jump drives from the Quarg. Also found out she was ambidextrous."

Author: Lemuria made a point of keeping her ambidextrous, so I'm moving it to the person log rather than the captain's log.

		event "beyond the hyperdrive payment" 40
		set "beyond the hyperdrive news can show now"
	`The moment you land, Anna is all too happy to see you. She grabs your shoulders with both arms and in an excited tone explains that her book has been approved for publication.`
	`	"They're publishing 'Beyond the Hyperdrive' today, and there's a full release event where I'll be signing copies! Thanks for everything you've done, <first>!"`
	choice
		`	"It got approved? That's awesome!"`
	`	"I know, right? So do you want to join me during the book signing?"`
	choice

> 		`	"I've been with you through the whole process. Count me in!"`

Reviewer 2: Two single issue choices in a row.

	`	"Great! Let's walk to the bookstore and get ready! The staff have already prepared seats for both of us."`

> 	`	You and Anna walk into the massive bookstore at the spaceport. There are already several Heliarch present, their Enforcer Riot Staffs in place for crowd control.`

Reviewer 2: ```suggestion
	`	You and Anna walk into the massive bookstore at the spaceport. There are already several Heliarchs present, their Enforcer Riot Staffs in place for crowd control.`
```

> 	`	A Kimek, dressed in what looks like the bookstore's uniform, takes out the Kimek equivalent of a loudspeaker and begins announcing the release of 'Beyond the Hyperdrive'. Within minutes, the bookstore has become crowded, with a long queue forming between the cashiers and your table, as Anna signs every copy brought to her by Kimek.`

Reviewer 2: 'Kimek equivalent of a loudspeaker'? Why not just a loudspeaker? What is it about this object that makes it different from a human loudspeaker? Not to sound harsh but '[alien] equivalent of [normal human object]' is kind of a lazy way of describing something.

	`	Occasionally, Anna alternates between her left and right hand to sign books whenever the other is tired. For hours, this continues. You tried to keep count of how many copies she signed, but gave up when it reached six hundred.`
	choice
		`	(Say nothing.)`
			goto continue
		`	"Are you left- or right-handed?"`
	`	"I was originally born left-handed, but as a child I chose to learn how to use my right hand, because I thought being able to use both hands would be cool."`
	label continue
	`	The signing continues for several more hours. Eventually, the same Kimek in the bookstore uniform announces that the store is closing soon.`
	`	As the store closes and the queue shortens, Anna signs the last Kimek's copy. With that, the event is over. Anna walks with you back to your ship to say goodbye.`

> 	`	"Today was a resounding success. My book succeeded beyond my wildest dreams, and it was all because of you <first>!" she says, almost in a teary voice. She pulls out a credit chip. "Here, <payment>. You've done so much for me and my career, and you deserve it. Additionally, you will be receiving a salary of 100 credits daily, a small but significant portion of my royalties."`

Reviewer 2: ```suggestion
	`	"My book succeeded beyond my wildest dreams, and it was all because of you, <first>!" she says, almost in a teary voice. She pulls out a credit chip. "Here, <payment>. You've done so much for me and my career, and you deserve it. Additionally, you will be receiving a salary of 100 credits daily, a small but significant portion of my royalties."`
```

> 	`	Anna then takes out another copy of "Beyond the Hyperdrive", which she signs in front of you. It is the Coalition edition of the book, but translated to your language. Quickly reading through it, you realize that the book has been significantly altered from the version sold in human space. "Here, take it. A special copy just for you, as a thank you for all you've done."`

Author: You were given the original version, remember?

> 	choice

Reviewer 2: Single issue choice.

	`	"As for the special copy, it was so I could verify and look at the changes they made, in my language, so I wouldn't have to learn their language. They agreed to print another copy for me so I could give it to you.`
	`	"I've got to go now. I want to see how well my book is selling. Goodbye!"`
	`	Anna then leaves your ship. You ponder what to do next, but you do feel rewarded having helped a struggling author make it big.`

event "beyond the hyperdrive payment"

mission "Beyond the Hyperdrive Gratitude Payment"
	landing
	source
		government "Coalition" "Heliarch"
	to offer
		has "event: beyond the hyperdrive payment"
	on offer
		payment 2000000
		conversation
			`You are shocked to see an extra <payment> in your account. Checking your messages you open a message from Anna, the author whose book 'Beyond the Hyperdrive' was published in Coalition space a few months ago. She explains that the book has been selling very well in Coalition space and sends an expression of gratitude. Given the <payment> you received this is likely an understatement.`

> 				decline

Reviewer 2: Does she just never go back to human space?

Author: I don't think so. I can ask Lemuria again, but I believe the intention was for her to stay.

Reviewer 3: > I don't think so. I can ask Lemuria again, but I believe the intention was for her to stay.

I never really fleshed that part of Fangvin out, so I'll leave that to you.

But I'm wondering what would happen to First Last if eventually they realize that Fangvin hasn't been seen in human space for a while. And then through investigations they see that Fangvin was last seen with First Last. Would that be a follow-up story? Who knows. As said before, I will leave it to you.

	source "Zug"
	destination "Ki Patek Ka"
	to offer
		has "Coalition: Struggling Author - Book Discovered: done"
	on offer
		conversation
			`Upon entering the spaceport, you walk past a restaurant. A woman raises her arms, waving towards you. She must be Anna Fangvin, the author the Coalition citizens showed interest in.`
			`	You approach the woman and sit next to her. "Hello, are you <first>?" she asks. "Yes," you answer. "What do you write?"`
			label "genre"
			`	"I write science fiction novels. I just released 'Beyond the Hyperdrive', the story of a group of humans who steal the Quarg's jumping technology and see what's truly beyond the hyperlanes we've known for so long. Here, take a look!"`
			`	Anna retrieves a hardcover book from her bag. The cover shows a human next to a Quarg Wardragon, and skimming the front and back flaps, you confirm most of what Anna told you about it. Her signature is on the title page.`
			choice
				`	"How long is the book?`
					goto "book length"
				`	"Why is it a story about humanity stealing from the Quarg?"`
					goto "quarg"
				`	"Anything else aside from Beyond the Hyperdrive?"`
			label "other novels"

> 			`	"Beyond the Hyperdrive is the fifth book I've ever published, actually," says Anna. "The other four never got any sales. I write short stories on the Hypernet in my free time. In fact, I'll probably start writing my sixth in a couple months, as soon as I find a good idea."`

Reviewer 4: According to the one precedent in vanilla ES, "hypernet" should be lowercase.

```suggestion
			`	"Beyond the Hyperdrive is the fifth book I've ever published, actually," says Anna. "The other four never got any sales. I write short stories on the hypernet in my free time. In fact, I'll probably start writing my sixth in a couple months, as soon as I find a good idea."`
```

					goto "quarg"
			label "book length"
			`	"About three hundred pages. Not too long."`
			choice
				`	"Have you written any other novels?"`
					goto "other novels"
				`	"Why is it a story about humanity stealing from the Quarg?"`
			label "quarg"
			`	"I have a lot of problems with them. I just don't trust them. I remember when my parents took me to Humanika. Naturally, I wanted to explore, so I wandered off. I was lost. This creepy-looking Quarg found me, and when it tried to approach, I cried.`
			`	"And then when I grew up, when I was a teenager, I decided to do a project on them for college. They responded with silence to my questions, as if they had something to hide. I worked as an assistant for a journalist for a few months. He kept asking me for help with the Quarg. They just would not budge."`
			choice
				`	"That must have been rough, stuck on Humanika. Did your parents eventually find you?"`
					goto "parents"
				`	"Well that's all well and good. But I have my own business to attend to now. Good luck with your book, thanks for your time, and goodbye."`
					decline
				`	(Tell her about the Coalition.)`
					goto "coalition"
			label "parents"

> 			`	"They eventually found me, thankfully. But it was distressing being stuck with this Quarg for 10 to 30 minutes, I don't remember how long, before eventually this other Quarg took my parents to me. I told them I wanted to get off Humanika as fast as possible and get back to Zug. They obliged, and we were off that world and I never wanted to come back."`

Reviewer 4: ```suggestion
			`	"They eventually found me, thankfully. But it was distressing being stuck with this Quarg for ten to thirty minutes, I don't remember how long, before eventually this other Quarg took my parents to me. I told them I wanted to get off Humanika as fast as possible and get back to Zug. They obliged, and we were off that world and I never wanted to come back."`
```

Words for these numbers is probably better, since technically the number 10 is low enough that it should be written out in full.

Last sentence is also a bit awkward, but it works if it's how Anna speaks.

Author: ```suggestion
			`	"They eventually found me, thankfully. But it was distressing being stuck with this Quarg for ten to thirty minutes, I don't remember how long, before eventually this other Quarg took my parents to me. I told them I wanted to get off Humanika as fast as possible and get back to Zug. They obliged, and we were off that world; I never wanted to come back."`
```

			choice

> 				`	"So you said your book doesn't sell well, right?"`

Reviewer 4: I think one of the following would be clearer, depending on your intended meaning:

```suggestion
				`	"So you said your books don't sell well, right?"`
```

```suggestion
				`	"So you said your book hasn't sold well, right?"`
```

```suggestion
				`	"So you said your book didn't sell well, right?"`
```

			`	"No, it doesn't. Do you know anyone who could help? My friends don't seem to be inclined to do anything for me."`
			choice
				`	"No actually, I'm afraid I don't know anyone who could."`
					decline
				`	(Tell her about the Coalition.)`
			label "coalition"

> 			`	You begin explaining the Coalition to Anna, going in-depth about their history and their culture, and their long history of opposing the Quarg, seeing as Anna has less than positive opinions of them. At first Anna does not believe you; but you reassure her it's real. You also mention the population of the Coalition and the novelty of an 'alien' author coming into the area.`

Reviewer 4: ```suggestion
			`	You begin explaining the Coalition to Anna, going in-depth about their history and their culture, and their long history of opposing the Quarg, seeing as Anna has less than positive opinions of them. At first Anna does not believe you; but you reassure her it's real. You also mention the population of the Coalition and the novelty of an "alien" author coming into the area.`
```

			`	"Will they really like my books?"`
			choice
				`	"They will, hopefully."`
				`	"Well, it's better to try than to just give up."`
			`	"These stories might just inspire my next novel. But, they wouldn't speak our language. How are they going to read my books?"`
			choice
				`	"They have a translation device."`
				`	"I'm sure they can learn the language."`

> 			`	"That's reassuring... but, how am I even going to get there?" she queries.`

Reviewer 4: It's best not to have choices interspersed with single lines of dialogue. Could you maybe cut one of these and just tell the player what they say? Something like "You explain that they have a translation device which allows them to communicate with humans."

			choice
				`	"I'll take you there."`
				`	"I don't think you can."`
					goto decline
			`	"Do I have to pay anything? As a struggling author, I can't afford off-world transport at all, let alone to a completely different alien race! Actually, I know - I'll give you a copy of Beyond the Hyperdrive! Thank you so much! Just wait for a while. I'll go over to my house and bring what I'll need for the trip, and then we can go to whatever this Coalition thing is," says Anna.`
			`	As you wait for Anna, you look over your maps of Coalition space to determine the most suitable place for her. After much deliberation and thought, you decide on <destination>.`
			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open the hatch to let her inside, as you lead her to her bunk and inform her of the destination. You prepare to leave the spaceport.`
				accept
			label decline

> 			`	Filled with disappointment, Anna says, "Well, I'm afraid I have other things to do than. At least thank you for the stories." She leaves.`

Reviewer 4: ```suggestion
			`	Filled with disappointment, Anna says, "Well, I'm afraid I have other things to do, then. At least thank you for the stories." She leaves.`
```

			`	"Okay," says Anna, "I'll explore the spaceport a little bit to find some inspiration. Get your ship ready and meet me there."`

mission "Coalition: Struggling Author 2"
	name "Help Anna Register"
	description "Take Anna Fangvin to the <destination> so she can register with the Heliarchs."
	source "Ki Patek Ka"
	destination "Ring of Friendship"
	to offer
		has "Coalition: Struggling Author 1: done"
	on offer
		conversation
			`You walk through the spaceport, looking for Anna. She is typing away on her laptop at a table in the middle of a public square.`
			`	"Oh, hello again <first>!" Anna says. "Is the ship ready?"`
			`	Hearing an affirmative answer, Anna stands up from her chair and begins following you to the ship. She settles into her bunk as you prepare for a trip to the <destination>.`
				accept
	on enter "Quaru"
		conversation
			`As you enter <system>, Anna walks out of her bunk and finds a spot on the side of the bridge, peering into the empty void. "What a massive ringworld!" she exclaims.`
			choice
				`	"It's enormous. As you know, they stole it from the Quarg roughly seven thousand years ago."`
				`	(Say nothing.)`

> 			`	"This is going to be a great source of inspiration for my next novel," says Anna, still in awe.`

Reviewer 4: Again, this is a bit short. Maybe another paragraph about how she keeps staring at it/starts typing or writing/does something else/goes back to sleep?

			`Going through the spaceport looking for Anna, you finally find her at a restaurant serving food made for Saryds. She's typing away furiously at her laptop, with a plate of vegetables next to her. She sees you and motions for you to take a seat.`
			`	You sit down next to Anna. "These vegetables are so delicious! After months of instant noodles and takeout back home, it finally feels good to eat something that's actually healthy." she exclaims, closing the computer.`
			choice
				`	"How good is it?"`
					goto good
				`	"What did you usually eat back in human space?"`
			`	"My diet wasn't exactly the best. All I had was carbs and takeout food; I never could find time in my schedule to cook at home. I was working hard on 'Beyond the Hyperdrive' at the time. This food is better than anything I could ever find on Zug."`
			choice
				`	"What's our plan now?"`
					goto published
				`	"What are we going to do next?"`
					goto published
			label good
			`	"It's very good. Probably better than anything I've had back on Zug. Now, I'd like to get my book published here."`
				goto company
			label published
			`	"I want to get my book published here."`
			label company

> 			`	"Have you found a publishing company yet?" you ask. "No." Anna then begins complaining about how it was extremely difficult for her to communicate with the restaurant staff, who did not have a translation box.`

Reviewer 4: ```suggestion
			`	"Have you found a publishing company yet?" you ask.`
			`"No," Anna replies, then begins complaining about how it was extremely difficult for her to communicate with the restaurant staff, who did not have a translation box.`
```

Otherwise it makes it look like the player says no.

			`	"Greetings, human. To be of assistance to you, how can I?" the clerk says.`
			choice
				`	"Hello. I am <first> <last>. This is my friend, Anna Fangvin. She is an author who wants to publish a book in Coalition space and needs a translation box to converse with the publishing company."`
			`	"A translator, the famous human needs?" asks the clerk.`
			choice
				`	"Yes."`
			`	"Trust you with a translator, we still cannot. And help you, I cannot, so a Heliarch, I must call," says the clerk, then proceeding to call for a Heliarch in its native language.`
			`	After a few minutes, the Heliarch approaches you and directs you into a room.`
			choice
				`	"Hello."`
			`	"Told me about your intentions, the clerk already has, and understand it, I do. Accompany you to the publishers, me and my agents will, once you decide upon a company."`
			`	"We need to find a publishing company then!" says Anna.`
			choice
				`	"The Kimek might be a good choice. They have a high population."`
					goto publishingcompany

> 				`	"Perhaps the Saryd. People with human parts are likely to be more interested in human books."`

Reviewer 4: Gah.

This is too many choices, and yet none at all. If there's only one option, and especially if that option is just a single word, please just tell me what I'm saying (either through dialogue or by describing what I do), instead of giving me buttons to click.

Author: I think I've resolved this, if you don't mind taking a look.

Reviewer 4: I think this mostly works, but the line about "what's our plan now?" could maybe be dechoiced as well, since Anna has a one-liner after it and then there's another choice.

			`	"I don't think I like the idea of selling to centaurs. Are there other options?"`
			choice

> 				`	"The Arachi have a close-knitted population. Word of a new book might spread quickly there."`

Reviewer 4: ```suggestion
				`	"The Arachi have a close-knit population. Word of a new book might spread quickly there."`
```

			`	"Required to stay with Anna, you are not, but if you wish to, stay with her you may. To return to human space, you are free to do," says the Kimek.`
				goto "rest of interview"
			label "lounge"
			`	You leave the room and walk to the lounge. There are several tables where many Kimek are chatting in their native language, while a few Saryds and Arachi are visibly relaxed. You sit at an empty table and pass the time, occasionally going to the water dispenser to hydrate yourself.`
			`	After thirty minutes, a Kimek approaches you. Its translation device speaks. "So, the alien in our space, you are?"`
			# kst-1 is short for [K]imek [s]mall [t]alk 1
			choice
				`	"I don't want to talk to you."`
					goto "kst-1 ignore"
				`	"Yes, I am <first> <last>. Glad to meet you."`
				`	"Well, I'm not the only alien."`
					goto "kst-1 not only alien"
				`	"We call ourselves humans, not aliens."`
					goto "kst-1 endonym"
			label "kst-1 glad to meet you"
			`	"Same. 'Akacharra', my name is. To our fine neighborhood, for what purpose have you come here?" "My friend Anna Fangvin is publishing a book here," you say.`
				goto "kst-1 anna"
			label "kst-1 not only alien"

> 			`	"If more of you, here there are, then who are they?" "It's just two; my friend Anna Fangvin and I. She's publishing a book in your space," you reply.`

Reviewer 4: You should probably add a paragraph break between the Kimek's words and the player's (same on later lines too).

				goto "kst-1 anna"
			label "kst-1 endonym"
			`	"'Humans', you are? Know that, I never did. For correcting me and educating me on your fellow kind, I am thankful," says the Kimek. "So, the other human, who is she?" "Oh, that's my friend Anna Fangvin. She's here to publish her book in Coalition space," you answer.`
			label "kst-1 anna"
			`	"Anna Fangvin, her name is? An interesting name, it is, and her book, just as interesting it will be when it is released. But know the title of the book, I do not, so if tell me you will, appreciate it I will."`
			choice
				`	"It is called 'Beyond the Hyperdrive'. It could very well be years before release, but I'm glad you're taking interest."`
				`	"It is called 'Beyond the Hyperdrive'. I think it's likely to be one of the best books of all time."`
			`	"So, 'Beyond the Hyperdrive', it is, you say? Thankful, I am, and on its release, buy the book I will remember to."`
			`	The Kimek walks away after the two of you exchange farewells, leaving you to your own devices.`
				goto "anna approaches"
			label "kst-1 ignore"

> 			`	"Apologies for invading your personal space, I give." The Kimek quickly walks away.`

Reviewer 4: Is this a logical response? Would the Kimek take the phrase "I don't want to talk to you" neutrally?

(This is a genuine question rather than criticism, and possibly one for Arachi.)

Author: I'm not well-versed enough in Coalition ideals to know. @Arachi-Lover, do you have thoughts on this?

	on offer
		conversation
			`Checking your computer feed for the latest galactic news and trade prices, you get a reminder.`
			`	You read it, and recall that you had helped an author named Anna Fangvin get her book, 'Beyond the Hyperdrive,' published about half a year ago in Coalition space. You should return to <destination> and check on her.`
				accept
	on complete
		payment 50000
		"salary: Beyond the Hyperdrive" = 100
		conversation "anna invites you to book signing"

mission "Coalition: Struggling Author 5B"
	landing
	source
		near "Quaru" 0 100
	destination "Inmost Blue"
	to offer
		has "event: beyond the hyperdrive is published"
		not "Coalition: Struggling Author 5A: offered"
	on offer
		conversation
			`As you land on the spaceport, you check your messages. You see a message from Anna Fangvin, the author who you helped about half a year ago. In the message, she explains that her novel, 'Beyond the Hyperdrive,' has been approved. You should return to <destination> and meet her.`
				accept
	on complete
		payment 50000
		"salary: Beyond the Hyperdrive" = 100
		conversation "anna invites you to book signing"

conversation "anna invites you to book signing"
	action

> 		log "People" "Anna Fangvin" `Anna is an ambidextrous science fiction writer. She was a struggling author in human space, but found success upon publishing her novel 'Beyond the Hyperdrive' in Coalition space. Her book is a bestseller in Kimek space.`

Reviewer 4: So many spaces!

```suggestion
		log "People" "Anna Fangvin" `Anna is an ambidextrous science fiction writer. A struggling author in human space, she found success upon publishing her novel 'Beyond the Hyperdrive' in the Coalition. Her book is a bestseller among Kimeks.`
```

	label continue
	`	The signing continues for several more hours. Eventually, the same Kimek in the bookstore uniform announces that the store is closing soon.`
	`	As the store closes and the queue shortens, Anna signs the last Kimek's copy. With that, the event is over. Anna walks with you back to your ship to say goodbye.`
	`	"My book succeeded beyond my wildest dreams, and it was all because of you, <first>!" she says, almost in a teary voice. She pulls out a credit chip. "Here, <payment>. You've done so much for me and my career, and you deserve it. Additionally, you will be receiving a salary of 100 credits daily, a small but significant portion of my royalties."`
	`	Anna then takes out another copy of "Beyond the Hyperdrive", which she signs in front of you. It is the Coalition edition of the book, but translated to your language. Quickly reading through it, you realize that the book has been significantly altered from the version you read in human space. "Here, take it. A special copy just for you, as a thank you for all you've done."`
	choice
		`	"Why is this book so different from the one in human space?"`
		`	"Did you make any changes to it?"`

> 	`	"Well, for one, it's not in our language. Some concepts just don't translate very well, so I tried to capture the spirit of the story. The Heliarch linguists were very helpful," she explains. "They really went into depth on some aspects of the Coalition's languages. None of the major plot points were changed, thankfully, though some content in the book had to be edited or removed. My Heliarch friend said that if I changed this or removed that, the book would sell so much better, and as they said, they wanted to 'make it more suitable for the Coalition'. Their words.`

Reviewer 4: ```suggestion
	`	"Well, for one, it's not in our language. Some concepts just don't translate very well, so I tried to capture the spirit of the story. The Heliarch linguists were very helpful," she explains. "They really went into depth on some aspects of the Coalition's languages. None of the major plot points were changed, thankfully, though some content in the book had to be edited or removed. My Heliarch friend said that if I changed this or removed that, the book would sell so much better, and as they said, they wanted to 'make it more suitable for the Coalition.' Their words.`
```

	`	"As for the special copy, it was so I could verify and look at the changes they made, in my language, so I wouldn't have to learn their language. They agreed to print another copy for me so I could give it to you.`
	`	"I've got to go now. I want to see how well my book is selling. Goodbye!"`

> 	`	Anna then leaves your ship. You ponder what to do next, but you do feel rewarded having helped a struggling author make it big.`

Reviewer 4: Don't tell me how I feel.

How about this?

```suggestion
	`	Anna then leaves your ship, leaving you with a novel, a payment, and the knowledge that you've helped a struggling author make it big.`
```

event "beyond the hyperdrive payment"

mission "Beyond the Hyperdrive Gratitude Payment"
	landing
	source
		government "Coalition" "Heliarch"
	to offer
		has "event: beyond the hyperdrive payment"
	on offer
		payment 2000000
		conversation

> 			`You are shocked to see an extra <payment> in your account. Checking your messages you open a message from Anna, the author whose book 'Beyond the Hyperdrive' was published in Coalition space a few months ago. She explains that the book has been selling very well in Coalition space and sends an expression of gratitude. Given the <payment> you received this is likely an understatement.`

Reviewer 4: "Checking your messages you open a message" is a little awkward. Maybe "checking your communicator/phone/portable 31st century fax machine that's less advanced than current human technology" instead?

			label published
			`	"On another note, I want to get my book published here."`
			label company
			`	"Have you found a publishing company yet?" you ask.`
			`	"No," Anna replies, then begins complaining about how it was extremely difficult for her to communicate with the restaurant staff, who did not have a translation box.`
			choice
				`	"I can get you a translator. It shouldn't cost anything."`
				`	"I'd be happy to help obtain a translator. I'm guessing we can get one for free."`
			`	"Great! So, where will we get the translator?"`
			`	"We'll go to <destination> to ask the Heliarchs for a translation box," you tell Anna. "Finish your meal, then go back to my ship. I'll be ready." You walk off to your ship.`
			`	After a few minutes, you see Anna walking towards you. You open the hatch as she enters. She settles into her bunk as you prepare to depart.`
				accept
	on complete
		conversation
			`You arrive at the spaceport. Anna leaves her bunk and waits for you to open the hatch. You quickly lead her to a Heliarch office. You walk to the counter, and begin conversing with the clerk.`
			`	"Greetings, human. To be of assistance to you, how can I?" the clerk says.`

> 			`	Anna speaks up. "I'm Anna Fangvin, and I'd like to publish a book here in Coalition space. My friend <first> <last> tells me I will need a translation box to converse with the publishing company. Could you provide one?"`

Reviewer: The publishing company would have a translator, I'd suggest altering the focus of this to being about Anna looking up different publishers and finding one that seemed more promising for her books' stories.

There can still be the Heliarch assisting you and Anna, would be kinda like them keeping an eye on it all, it's just the focus on getting the translator right away for her that I'd shift, as the player should know it's not that simple to get one.

			`	"A translator, the human needs?" asks the clerk.`
			choice
				`	"Yes."`
			`	"Trust you with a translator, we still cannot. And help you, I cannot, so a Heliarch, I must call," says the clerk, then proceeding to call for a Heliarch in its native language.`
			`	After a few minutes, the Heliarch approaches you and directs you into a room.`
			`	"Told me about your intentions, the clerk already has, and understand it, I do. Accompany you to the publishers, me and my agents will, once you decide upon a company."`
			`	"We need to find a publishing company then!" says Anna.`
			choice
				`	"The Kimek might be a good choice. They have a high population."`
					goto publishingcompany
				`	"Perhaps the Saryd. People with human parts are likely to be more interested in human books."`

> 			`	"I don't think I like the idea of selling to centaurs. Are there other options?"`

Reviewer: ```suggestion
			`	"I thought the same at first, but from what I've seen the Saryds seem pretty reserved, don't you think? Maybe another species would be better."`
```

			choice
				`	"The Arachi have a close-knit population. Word of a new book might spread quickly there."`
				`	"The Kimek have the most people of any species in Coalition space. Let's try selling to them."`
					goto publishingcompany

> 			`	"Spiders reading my book just sounds creepy. Could we do the Kimek instead? They have a high population, right?"`

Reviewer: ```suggestion
			`	"Oh, it's kind of embarrassing but... I'm actually a bit afraid of spiders, so I'm going to take a while to get used to the Arachi. Perhaps the Kimek would be easier? They have a high population, right?"`
```

	name "Look for Anna Fangvin"
	description "Get in contact with Anna Fangvin, an author in which the Coalition book club has shown interest in."
	source
		government "Coalition"
	destination "Zug"
	to offer
		has "event: coalition citizens discover fangvin"
	on offer
		conversation
			`Upon landing in the spaceport, you get a message from the same group of Coalition citizens. You recall that you had provided them with a datachip containing a copy of Alexandria's archive.`
			``
			`	Dear <first>,`
			`		Interested in an author from the datachip, we are. "Anna Fangvin," her name is. While the full book, access it we could not, the summary, about your race's actions against the oppressors, captivate us it did. To converse with her, we wish, and a copy of the book, we desire.`
			choice
				`	(Look for Anna Fangvin.)`
				`	(Discard the message.)`
					decline
			`	A quick look through your encyclopedia reveals nothing. However, checking a copy of the data chip that you gave the Coalition citizens reveals Anna Fangvin to be an author from <destination>. You decide to meet her there for an interview.`
				accept
	on complete
		conversation

> 			`As you prepare to land, you write Mrs. Fangvin a short message, offering an interview. In less than ten minutes you receive a reply, explaining that she has accepted your offer and will wait for you in the spaceport.`

Author: I had planned on making that `Ms.` but as you'd prefer Anna I'll do that.

	choice
		`	"Why is this book so different from the one in human space?"`
		`	"Did you make any changes to it?"`
	`	"Well, for one, it's not in our language. Some concepts just don't translate very well, so I tried to capture the spirit of the story. The Heliarch linguists were very helpful," she explains. "They really went into depth on some aspects of the Coalition's languages. None of the major plot points were changed, thankfully, though some content in the book had to be edited or removed. My Heliarch friend said that if I changed this or removed that, the book would sell so much better, and as they said, they wanted to 'make it more suitable for the Coalition.' Their words.`
	`	"As for the special copy, it was so I could verify and look at the changes they made, in my language, so I wouldn't have to learn their language. They agreed to print another copy for me so I could give it to you.`

> 	`	"I've got to go now. I want to see how well my book is selling. Goodbye!"`

Reviewer: Maybe mention her also being excited to find out about the reception to her book? Reading online about it or just talking with others on how they like it.

	`	Anna then leaves your ship, leaving you with a novel, a payment, and the knowledge that you've helped a struggling author make it big.`

> outfit "Beyond the Hyperdrive"

Reviewer: This outfit should be in a separate file, either `outfits.txt` in the human directory, or `coalition outfits.txt` in the Coalition directory.

	category "Unique"
	series "Non-Functional Unique"
	index 11
	thumbnail "outfit/beyond the hyperdrive"
	"unique" 1

> 	description "A book gifted to you by Anna Fangvin, a struggling author who found success in Coalition space."

Reviewer: It would be cool if the book description had something of a simple summary of the story, as another description paragraph. Here's kind of what I'm thinking, using The Hobbit as an example:

```
	outfit "The Hobbit"
		description "A book gifted to you by the ghost of J.R.R. Tolkien, who manifested within your ship one fateful night."
		description "	The story follows a hobbit named Bilbo, who embarks in an adventure with a group of dwarves seeking to reclaim their lost gold, in a mountain captured by a dragon. After a long and arduous journey, many twists and turns, ups and downs, and barely surviving goblins, spiders and a massive dragon, Bilbo returns to his homely hobbit hole, with a chest full of treasure and a life's worth of adventures."
```

@a-random-lemurian did you have an idea/summary sort of like this for the story of the book?

Author: Thinking something like `The story follows a human captain who steals a Jump Drive from the Quarg and travels beyond the hyperlanes, learning new things about aliens all around us.`
Could also add the captain's name there as a joke.

Reviewer 5: I like the Captain's name idea, here's one take:

`The story follows a human, Captain <first> Standin, who steals a Jump Drive from the Quarg and travels beyond the hyperlanes, learning new things about aliens all around us.`

Author: Although - do substitutions work in outfits? Probably worth checking that.

![Screen Shot 2025-04-07 at 6 15 47 AM](https://github.com/user-attachments/assets/6efc6f9b-57ac-4d72-81dc-2160dd329297)

Unfortunately I don't think we'll be able to do that.

Reviewer 3: > It would be cool if the book description had something of a simple summary of the story, as another description paragraph. Here's kind of what I'm thinking, using The Hobbit as an example:
> 
> ```
> 	outfit "The Hobbit"
> 		description "A book gifted to you by the ghost of J.R.R. Tolkien, who manifested within your ship one fateful night."
> 		description "	The story follows a hobbit named Bilbo, who embarks in an adventure with a group of dwarves seeking to reclaim their lost gold, in a mountain captured by a dragon. After a long and arduous journey, many twists and turns, ups and downs, and barely surviving goblins, spiders and a massive dragon, Bilbo returns to his homely hobbit hole, with a chest full of treasure and a life's worth of adventures."
> ```
> 
> @a-random-lemurian did you have an idea/summary sort of like this for the story of the book?

I never had a "Wikipedia plot section"-length summary of the book. The one sentence we have is good enough, I guess.

Author: Well, I've added a short summary. Now that we have it, I don't think there's a reason to remove it.

		payment 5000000
		fail

mission "Coalition: Struggling Author - Human Media 1"
	minor
	name "Give Kachachee Cultural Information"
	description "A group of Coalition citizens have asked you for information on the human race."
	source "Ki Patek Ka"
	stopover "Alexandria"
	destination "Ki Patek Ka"
	to offer
		has "First Contact: Coalition: done"
	on offer
		conversation
			`As you walk through the spaceport, you see a group of Kimek and an Arachi seated at a table, deep in conversation. One of them glances toward you. As if something suddenly clicked, the group quickly rises from their seats and begin approaching you.`
			choice
				`	"Excuse me, what do you all want?"`
					goto "literature collectors"
				`	"Hello? What is it?"`
					goto "literature collectors"
				`	(Reach for your gun.)`
			`	The lead Kimek jumps back. "Mean you harm, we did not. Unrelated to violence, our quest is. Speak to you, may we?"`
			choice
				`	"What do you want?"`
					goto "literature collectors violent"
				`	"No, you may not. Goodbye."`
			`	The group of Coalition citizens go back to their table with disappointed expressions.`
				decline
			label "literature collectors violent"

> 			`	"Collectors of literature, we are. Curious about your race, and the books they have written, we are, so for you, a mission we have," says the lead Kimek, who then introduces themselves as Kachachee. "Literature of many kinds, we read, from science fiction to tales of alien worlds outside of our homeland.`

Reviewer: That stopped being the case the moment the Kimek made first contact with the Quarg.

Their literature regarding tales of aliens outside their homeland includes at least: the Quarg, the Saryds, the Arachi, the humans.

mission "Coalition: Struggling Author - Book Discovered"
	name "Look for Anna Fangvin"
	description "Get in contact with Anna Fangvin, an author that the Coalition book club has shown interest in."
	source
		government "Coalition"
	destination "Zug"
	to offer
		has "event: coalition citizens discover fangvin"
	on offer
		conversation
			`Upon landing in the spaceport, you get a message from the group of Coalition citizens that you had provided with a datachip containing a copy of Alexandria's archive:`
			``
			`	Dear <first>,`
			`		Interested in an author from the datachip, we are. "Anna Fangvin," her name is. While the full book, access it we could not, the summary, about your race's actions against the oppressors, captivate us it did. To converse with her, we wish, and a copy of the book, we desire.`
			choice
				`	(Look for Anna Fangvin.)`
				`	(Discard the message.)`
					decline
			`	A quick look through your encyclopedia reveals nothing. However, checking a copy of the data chip that you gave the Coalition citizens reveals Anna Fangvin to be an author from <destination>. You decide to meet her there for an interview.`
				accept
	on complete
		conversation

> 			`As you prepare to land, you send Anna a short message asking to interview her at a restaurant near the spaceport. Less than ten minutes later, she replies - she accepted your offer and will wait for you in the spaceport.`

Author: If the first mission has a requirement that you've either not joined or have finished the main plot, this one could still potentially offer during FW. Locking missions behind the main plot altogether doesn't seem wise.
I don't imagine Anna would be affected too much by the bombing.

			`	"<first>! I found a company who's willing to publish me! I sent them a digital copy of my book and they instructed me to visit their office on <destination>," explains Anna, once again enthusiastic.`
			choice
				`	"Great!"`
					goto go
				`	"Let's go!"`
					goto go
				`	"What's the publishing company called?"`
			`	"They're called 'Priyepichia Publishing.'" Anna shows you beautiful pictures of a clean white building perched on a mountainside, high above gentle waves. On the roof stands a landing pad painted with a blue dolphin.`
			label go
			`	Anna heads to your ship, and you plot a course to <destination>. Just before you launch, the Heliarch hops in.`
				launch
	on complete
		event "beyond the hyperdrive is published" 165 197
		conversation
			`Landing on <planet>, Anna is excitedly holding a copy of 'Beyond the Hyperdrive.' She hastily opens the hatch as soon as it unlocks.`
			`	Anna stops, waiting for you to catch up. "In the message they sent me, they included instructions on how to reach their offices," she explains. "Let's go!"`

> 			`	You begin following Anna to the publisher's offices. You walk through the spaceport and its endless rows of shops and streets, eventually approaching what looks like a train station. Anna looks at the message again and tells you which platform to go to and which train to take. After a few minutes the train arrives. You and Anna board the train and travel to a neighborhood, with a large white building dominating the skyline.`

Reviewer 5: ```suggestion
			`	You begin following Anna to the publisher's offices. You walk through the spaceport and its endless rows of shops and streets, eventually approaching what looks like a train station. Anna looks at the message again and tells you which platform to go to and which train to take. After a few minutes the train arrives. You and Anna board the train and travel to a neighborhood where a large white building dominates the skyline.`
```

> 			`	Once the train stops, you get off onto the platform and out of the station. You walk around the building and find an entrance, busy with many Kimek and the occasional Saryd and Arach entering and exiting. Anna looks for a front desk and directs you to walk to a Kimek who looks like a receptionist.`

Reviewer 5: ```suggestion
			`	Once the train stops, you get off onto the platform and exit the station. Circling around to the publisher's office, you find the entrance bustling with Kimek, along with the occasional Saryd and Arach coming and going. Anna looks for a front desk and directs you to a Kimek who looks appears to be a receptionist.`
```

			choice
				`	"Hello, do you happen to know where Priyepichia Publishing is?"`
				`	"We're looking for a publishing company called Priyepichia Publishing. Could you provide directions?"`

> 			`	"On the 16th floor, Priyepichia Publishing is. Over there, the elevators are," says the receptionist's translator.`

Reviewer 5: Think this chunk is a _bit_ too long. Maybe add a choice to exit the elevator?
![image](https://github.com/user-attachments/assets/979c4fdd-ba14-40a0-9589-3e5ed370dc79)

			`	You turn to where the Kimek points you and see six large elevators, larger than what you are familiar with in human space. You look at all six and choose to enter the least crowded one.`
			`	You and Anna enter the elevator. You hear the noises of the Kimek communicating in their native language and get the feeling they're talking about you.`
			`	The elevator makes many stops at each floor of the building, as more and more Kimek reach their destinations. When the door opens on the 16th floor, the elevator is now half-full; and before you is a Kimek wearing an ornate jeweled necklace as well as a translation device.`
			`	"Greetings, humans. Follow me, you must, so discuss your book, we can," the Kimek says through its translation box.`
			`	At the direction of the Kimek, you and Anna walk as directed until you arrive at a room with glass walls. Along the way, you pass a lounge, a conference area where multiple Kimek are discussing something at a table with a group of Heliarchs, and a large open workspace where individual Kimek sit at their own computers.`

> 			`	After a few minutes of walking, there is an office where another Kimek and a team of five Heliarchs are sitting. The Kimek directing you points you towards that office and walks over to speak with the group.`

Reviewer 5: Describing the Heliarch species the first time they're seen, and more thematic with equal Heliarch representation from each species
```suggestion
			`	After a few minutes of walking, there is an office where another Kimek sits with a team of six Heliarchs - two from each species. The Kimek guiding you gestures toward the office, and heads over to speak with the group.`
```

> 			`	As you enter, Anna puts down the copy of her book on the table, and you each take a seat. The Kimek that led you here briefly speaks to the other Kimek sitting at the interview table before leaving.`

Reviewer 5: ```suggestion
			`	As you enter, Anna puts down the copy of her book on the table, and you each take a seat. The Kimek that led you here briefly speaks to the other non-Heliarch Kimek sitting at the interview table before leaving.`
```

				`	"Should I stay here to accompany Anna?`
					goto "accompany"
			label "wait time"
			`	"Wait a few hours, you must, but for the entire process, perhaps half a year, wait Anna must," explains Ikiyachee.`
			choice
				`	"A few hours? That's quite boring, isn't it?"`
					goto "boring"
				`	"Should I stay here to accompany Anna?"`
					goto "accompany"
			label "boring"
			`	"If boring for you, the interview is, then leave you may, to the lounge."`
			choice
				`	"Then I'm going to the lounge. Where is it?"`
				`	(Accompany Anna during the interview.)`
					goto "stay in interview"
			label "directions to lounge"
			`	Ikiyachee gives you directions to the lounge, and you head there to wait out the interview.`
				goto "lounge"
			label "accompany"
			`	"To stay with her during the interview, we allow you to, but a lounge for you, we have. If too boring, the interview is, then leave you may."`
			choice
				`	(Wait in the lounge.)`
					goto "directions to lounge"
				`	(Accompany Anna during the interview.)`
			label "stay in interview"
			`	Once the first questions are asked, Anna is all too happy to begin discussing the contents of her novel in great detail.`

> 			`	Under the watchful eye of the two Heliarchs in the room, Anna explains her anti-Quarg sentiments, the reasoning behind her choice to write the book, and the content of the book. She speaks quickly, but the Heliarchs' numerous questions for clarification and elaboration make the process a long one.`

Reviewer 5: Aren't there five Heliarchs in the room (six if you use the above suggestion)?
```suggestion
			`	Under the watchful eye of the six Heliarchs in the room, Anna explains her anti-Quarg sentiments, the reasoning behind her choice to write the book, and the content of the book. She speaks quickly, but the Heliarchs' numerous questions for clarification and elaboration make the process a long one.`
```

				goto "anna approaches"
			label "kst-1 ignore"
			`	"Apologies for invading your personal space, I give." The Kimek quickly walks away.`
			label "anna approaches"
			`	After a few hours, you see Anna enter the lounge and walk toward you. She takes a seat next to you.`
			choice
				`	"How did the interview go?"`
			`	"They told me that due to the review process to make sure it's suitable, I'll have to wait for roughly half a year. Thankfully, they have arranged a small apartment for me to live in. You can go on your own now, but please make sure you come back. If you want to return to human space, please remember to come back and meet me."`
			choice
				`	"Why don't you just go back to human space?"`
					goto stay
				`	"I'll make sure to do that."`
			label thanks
			`	"Thanks again, <first> <last>. I'll forever be grateful to you for helping me. I never thought I'd be successful as an author, but when I met you, you helped me turn my career into something I thought I'd never see."`
			choice
				`	"Just helping out."`
				`	"I'm happy to have helped you."`
			`	"Again, thank you. You will not be needed anymore for the process."`
			choice
				`	(Leave.)`

> 			`	You return to the spaceport the way you came and return to your ship. You are slightly anxious about the outcome of the process and whether Anna's book will even make it through the process to be published, but you have more important things to attend to for the next six months.`

Reviewer 5: Don't tell the player how anxious they are.
```suggestion
			`	You return to the spaceport the way you came and return to your ship. You have more important things to attend to for the next six months than wonder whether Anna's book will make it through the process to be published.`
```

				decline
			label stay
			`	"I need to stay here so I can answer their questions during the review process. Don't worry about me."`
			choice
				`	"Got it. Good luck publishing your book."`
					goto thanks

event "beyond the hyperdrive is published"

mission "Coalition: Struggling Author 5A"
	landing
	source
		not near "Quaru" 0 100
	destination "Inmost Blue"
	to offer
		has "event: beyond the hyperdrive is published"
		not "Coalition: Struggling Author 5B: offered"
	on offer
		conversation
			`Checking your computer feed for the latest galactic news and trade prices, you get a reminder.`
			`	You read it, and recall that you had helped an author named Anna Fangvin get her book, 'Beyond the Hyperdrive,' published about half a year ago in Coalition space. You should return to <destination> and check on her.`
				accept
	on complete
		payment 750000
		conversation "anna invites you to book signing"

> mission "Coalition: Struggling Author 5B"

Reviewer 5: Needs a proper name
![image](https://github.com/user-attachments/assets/6f14cbd8-7e01-4de9-bd90-6018709b905d)

	landing
	source
		near "4 Winter Rising" 1 100
	destination "Inmost Blue"
	to offer
		has "event: beyond the hyperdrive is published"
		not "Coalition: Struggling Author 5A: offered"
	on offer
		conversation
			`As you land on the spaceport, you check your messages and see a notification from Anna Fangvin, the author who you helped about half a year ago. In the message, she explains that her novel, "Beyond the Hyperdrive," has been approved. You should return to <destination> and meet her.`
				accept
	on complete
		payment 750000
		conversation "anna invites you to book signing"

conversation "anna invites you to book signing"
	action

> 		log "People" "Anna Fangvin" `Anna is an ambidextrous science fiction writer. A struggling author in human space, she found success upon publishing her novel 'Beyond the Hyperdrive' in the Coalition. Her book is a bestseller in the Kimek region.`

Reviewer 5: If you say nothing below you don't definitively learn she's ambidextrous. That also feels like a minor note that only comes up in the one line?
```suggestion
		log "People" "Anna Fangvin" `Anna was a struggling science fiction author in human space until she found success publishing her novel 'Beyond the Hyperdrive' in the Coalition. Her book is a bestseller in the Kimek region.`
```

	`	A Kimek, dressed in what looks like the bookstore's uniform, takes out a loudspeaker with multiple handholds and begins announcing the release of 'Beyond the Hyperdrive'. Within minutes, the bookstore has become crowded, with a long queue forming between the cashiers and your table, as Anna signs every copy brought to her.`
	`	Occasionally, Anna alternates between her left and right hand to sign books whenever the other is tired. For hours, this continues. You tried to keep count of how many copies she signed, but gave up when it reached six hundred.`
	choice
		`	(Say nothing.)`
			goto continue
		`	"Are you left- or right-handed?"`
	`	"I was originally born left-handed, but as a child I chose to learn how to use my right hand, because I thought being able to use both hands would be cool."`
	label continue
	`	The signing continues for several more hours. Eventually, the same Kimek in the bookstore uniform announces that the store is closing soon.`
	`	As the store closes and the queue shortens, Anna signs the last Kimek's copy. With that, the event is over. Anna walks with you back to your ship to say goodbye.`

> 	`	"My book succeeded beyond my wildest dreams, and it was all because of you, <first>!" she says, almost in a teary voice. She pulls out a credit chip. "Here, <payment>. You've done so much for me and my career, and you deserve it. Additionally, you will be receiving a salary of 100 credits daily, a small but significant portion of my royalties."`

Reviewer 5: Took out the salary line.
```suggestion
	`	"My book succeeded beyond my wildest dreams, and it was all because of you, <first>!" she says, almost in a teary voice. She pulls out a credit chip. "Here, <payment>. You've done so much for me and my career, and you deserve it."`
```

mission "Coalition: Struggling Author 1"
	name "Take Anna to Coalition Space"
	description "Take Anna Fangvin to <destination>, where she might find success as an author after having struggled on <origin> for so long."
	source "Zug"
	destination "Ki Patek Ka"
	passengers 1
	to offer
		has "Coalition: Struggling Author - Book Discovered: done"
	on offer
		conversation
			`Upon entering the spaceport, you walk past a restaurant. A woman raises her arms, waving towards you. She must be Anna Fangvin, the author the Coalition citizens showed interest in.`
			`	You approach the woman and sit next to her. "Hello, are you <first>?" she asks. "Yes," you answer. "What do you write?"`
			`	"I write science fiction novels. I just released 'Beyond the Hyperdrive', the story of a group of humans who steal the Quarg's jumping technology and see what's truly beyond the hyperlanes we've known for so long. Here, take a look!"`
			`	Anna retrieves a hardcover book from her bag. The cover shows a human next to a Quarg Wardragon, and skimming the front and back flaps, you confirm most of what Anna told you about it. Her signature is on the title page.`
			label "genre choice"
			choice

> 				`	"How long is the book?`

Author: I could change it to asking about words. The other possibility is that there aren't page numbers.

			choice
				`	"I'll take you there."`
				`	"I don't think you can."`
					goto decline
			`	"Do I have to pay anything? As a struggling author, I can't afford off-world transport at all, let alone to a completely different segment of the galaxy! Actually, I know - I'll give you a copy of Beyond the Hyperdrive! Thank you so much! Just wait for a while. I'll go over to my house and bring what I'll need for the trip, and then we can go to whatever this Coalition thing is," says Anna.`
			`	As you wait for Anna, you look over your maps of Coalition space to determine the most suitable place for her. After much deliberation and thought, you decide on <destination>.`
			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open the hatch to let her inside, as you lead her to her bunk and inform her of the destination. You prepare to leave the spaceport.`
				accept
			label decline
			`	Filled with disappointment, Anna says, "Well, I'm afraid I have other things to do, then. Thank you for the stories, at least." She leaves.`
				decline
	on complete
		set "coalition: struggling author - beyond the hyperdrive"
		conversation

> 			`Anna gives you the book, and then you leave the ship. The moment you are out, three Kimek approach you and Anna. The Kimek in the middle speaks, and then a brief silence occurs. It is broken as their translation box speaks.`

Reviewer 5: ```suggestion
			`Anna gives you the book before you both exit the <ship>. The moment you are out, three Kimek approach you and Anna. The Kimek in the middle speaks, and then a brief silence occurs. After a moment, their translation box breaks the silence.`
```

			`	"You have to register yourself with the Heliarchs before you can do much here. I'll take you to the Ring of Friendship, which is where you can complete the registration process." you say.`
			`	"Okay," says Anna, "I'll explore the spaceport a little bit to find some inspiration. Get your ship ready and meet me there."`

mission "Coalition: Struggling Author 2"
	name "Help Anna Register"
	description "Take Anna Fangvin to the <destination> so she can register with the Heliarchs."
	source "Ki Patek Ka"
	destination "Ring of Friendship"
	passengers 1
	to offer
		has "Coalition: Struggling Author 1: done"
	on offer
		conversation
			`You walk through the spaceport, looking for Anna. You find her in the middle of a public square, seated at a table with her laptop out. You figure she must've intended on getting started on some early work, what with the newly found inspiration of being in such an alien society, but it seems the sheer volume and diversity of peoples here on <origin> made her forget to type entirely.`
			`	She only seems to notice you when you also sit down beside her. "Oh, hello again, <first>!" she says, snapping back into focus. "Is the ship ready?"`

> 			`	Hearing an affirmative answer, Anna stands up from her chair and begins following you to the ship. She settles into her bunk as you prepare for a trip to the <destination>.`

Reviewer 5: Less repetitive occasionally using the ship name
```suggestion
			`	Hearing an affirmative answer, Anna stands up from her chair and begins following you to the <ship>. She settles into her bunk as you prepare for the trip to the <destination>.`
```

			choice
				`	"It's enormous. As you know, they stole it from the Quarg roughly seven thousand years ago."`
				`	(Say nothing.)`
			`	"This is going to be a great source of inspiration for my next novel," says Anna, still in awe. She pulls out a small notebook and begins busily writing in it.`
	on complete
		conversation
			`Anna is amazed as you touch down on the ringworld. As the two of you open the hatch, you see a Heliarch agent in front of you.`
			`	"Greetings, humans. For you, what can I do today?" asks the Heliarch. You respond that you would like to register Anna with the Heliarchy. After reaching the registration office, Anna enters it, coming out after fifteen minutes. "They finally got all the paperwork done. I'm off to the spaceport to get some food. Meet me there to discuss our next move."`

mission "Coalition: Struggling Author 3"
	name "Back to <planet>"
	description "Now that Anna Fangvin has completed the registration process, help her get a translation device on <destination>."
	source "Ring of Friendship"
	destination "Ki Patek Ka"
	passengers 1
	to offer
		has "Coalition: Struggling Author 2: done"
	on offer
		conversation

> 			`Going through the spaceport looking for Anna, you finally find her at a restaurant serving food made for Saryds. She's typing away furiously at her laptop, with a plate of vegetables next to her. She sees you and motions for you to take a seat.`

Reviewer 5: It could read as if Anna was serving the food otherwise.
```suggestion
			`Going through the spaceport looking for Anna, you finally find her at a restaurant, eating food made for Saryds. She's typing away furiously at her laptop with a plate of vegetables next to her. She sees you and motions for you to take a seat.`
```

			`	Once the train stops, you get off onto the platform and out of the station. You walk around the building and find an entrance, busy with many Kimek and the occasional Saryd and Arach entering and exiting. Anna looks for a front desk and directs you to walk to a Kimek who looks like a receptionist.`
			choice
				`	"Hello, do you happen to know where Priyepichia Publishing is?"`
				`	"We're looking for a publishing company called Priyepichia Publishing. Could you provide directions?"`
			`	"On the 16th floor, Priyepichia Publishing is. Over there, the elevators are," says the receptionist's translator.`
			`	You turn to where the Kimek points you and see six large elevators, larger than what you are familiar with in human space. You look at all six and choose to enter the least crowded one.`

> 			`	You and Anna enter the elevator. You hear the noises of the Kimek communicating in their native languages and get the feeling they're talking about you.`

Reviewer 5: Unless they hold conversations in multiple native Kimek languages?
```suggestion
			`	You and Anna enter the elevator. You hear the noises of the Kimek communicating in their native language and get the feeling they're talking about you.`
```

> 			`	The elevator makes many stops at each floor of the building, as more and more Kimek reach their destinations. When the door opens on the 16th floor, the elevator is now half-full; and before you is a Kimek with a necklace around it with ornate jewelry and a translation device.`

Reviewer 5: ```suggestion
			`	The elevator makes many stops at each floor of the building, as more and more Kimek reach their destinations. When the door opens on the 16th floor, the elevator is now half-full; and before you is a Kimek wearing an ornate jeweled necklace as well as a translation device.`
```

> 			`	"Greetings, humans. Follow me, you must, so discuss your book, we can," says the Kimek through its translation box.`

Reviewer 5: ```suggestion
			`	"Greetings, humans. Follow me, you must, so discuss your book, we can," the Kimek says through its translation box.`
```

> 			`	At the direction of the Kimek, you and Anna begin walking until directed to a room with glass walls. You walk by a lounge, an area where multiple Kimek are at a table discussing something with a group of Heliarchs, and a large open workspace where individual Kimek are sitting at their own computers.`

Reviewer 5: ```suggestion
			`	At the direction of the Kimek, you and Anna walk as directed into a room with glass walls. You walk by a lounge area, where multiple Kimek are conversing at a table with a group of Heliarchs, and a large open workspace where individual Kimek are sitting at their own computers.`
```

Author: Hmm, this suggestion changes the meaning. I think the original purpose was to have three separate rooms.

Reviewer 5: Ah, okay. How's
```suggestion
			`	At the direction of the Kimek, you and Anna walk as directed until you arrive at a room with glass walls. Along the way, you pass a lounge, a conference area where multiple Kimek are discussing something at a table with a group of Heliarchs, and a large open workspace where individual Kimek sit at their own computers.`
```

> 			`	After a few minutes of walking, there is an office with another Kimek and a team of five Heliarchs sitting. The Kimek directing you points toward that room and leaves.`

Reviewer 5: The leading Kimek can't leave twice (here and 3604)
```suggestion
			`	After a few minutes of walking, there is an office where another Kimek and a team of five Heliarchs are sitting. The Kimek directing you points you towards that office and walks over to speak with the group.`
```

> 			`	As you enter, Anna puts down the copy of her book on the table, and the two of you take a seat. The Kimek that led you here briefly speaks to the other Kimek sitting at the interview table, before leaving.`

Reviewer 5: ```suggestion
			`	As you enter, Anna puts down the copy of her book on the table, and you each take a seat. The Kimek that led you here briefly speaks to the other Kimek sitting at the interview table before leaving.`
```

> 			`	The Kimek's translation device begins to speak.`

Reviewer 5: For clarity
```suggestion
			`	The sitting Kimek's translation device begins speaking.`
```

				`	(Accompany Anna during the interview.)`
					goto "stay in interview"
			label "directions to lounge"
			`	Ikiyachee gives you directions to the lounge, and you head there to wait out the interview.`
				goto "lounge"
			label "accompany"
			`	"To stay with her during the interview, we allow you to, but a lounge for you, we have. If too boring, the interview is, then leave you may."`
			choice
				`	(Wait in the lounge.)`
					goto "directions to lounge"
				`	(Accompany Anna during the interview.)`
			label "stay in interview"
			`	Once the first questions are asked, Anna is all too happy to begin discussing the contents of her novel in great detail.`
			`	Under the watchful eye of the two Heliarchs in the room, Anna explains her anti-Quarg sentiments, the reasoning behind her choice to write the book, and the content of the book. She speaks quickly, but the Heliarchs' numerous questions for clarification and elaboration make the process a long one.`
			`	"Sold many books before coming here, have you?" one of the agents asks. Anna embarrassedly glances in your direction. "Well, no," she replies. "But I've been writing better and better over the past few years."`

> 			`	Priyepichia's representative speaks up. "Page count, we must know. Binding process, aware of you are? You want what type, and cover style?" "There are 300 pages. The hardcover book should have a Jump Drive emblazoned on the front cover.`

Reviewer 5: Wouldn't pages be totally different across Coalition vs humanity (what with different font sizes from vision and written language differences, size of book pages commonly used by Coalition vs human species, translations, and whatnot). I feel something like word count would be something slightly more meaningful at least. 
```suggestion
			`	Priyepichia's representative speaks up. "Word count, we must know. Binding process, aware of you are? You want what type, and cover style?" "There are around 90,000 words. The hardcover book should have a Jump Drive emblazoned on the front cover.`
```

			`	"I don't know what the binding type means; couldn't you just decide that yourself?"`
			`	As you listen, you learn that due to the Coalition's review process to make the book more suitable, Anna will have to stay for about half a year, during which she will have an apartment to live in.`
			choice
				`	"Do I have to stay with Anna here for half a year?"`
					goto "half a year"
				`	(Say nothing.)`
			label "rest of interview"

> 			`	As the hours pass, Anna's enthusiastic tone remains strong and constant. The Kimek then takes the book, puts it in a nearby filing cabinet and thanks you and Anna.`

Reviewer 5: Just grouped the two of you since it felt odd for them to thank you first for Anna's book.
```suggestion
			`	As the hours pass, Anna's enthusiastic tone remains strong and constant. The Kimek then takes the book, puts it in a nearby filing cabinet and thanks the two of you.`
```

> 			`	You leave the interview room, the Heliarchs staying in the room, presumably to continue discussion with the Kimek. As the two of you walk, Anna begins to speak.`

Reviewer 5: ```suggestion
			`	You leave the interview room, while the Heliarchs stay, presumably to continue speaking with the Kimek. As the two of you walk out, Anna begins to speak.`
```

				goto thanks
			label "half a year"
			`	"Required to stay with Anna, you are not, but if you wish to, stay with her you may. Return to human space, you are free to do," says the Kimek.`
				goto "rest of interview"
			label "lounge"

> 			`	You leave the room and walk to the lounge. There are several tables where many Kimek are chatting in their native language, while a few Saryds and Arachi are visibly relaxed. You sit at an empty table and pass the time, occasionally going to the water dispenser to hydrate yourself.`

Reviewer 5: ```suggestion
			`	You leave the room and walk to the lounge. There are several tables where many Kimek are chatting in their native language, while a few Saryds and Arachi are visibly relaxed. You sit at an empty table to pass the time, occasionally going to the water dispenser to hydrate yourself.`
```

				goto "anna approaches"
			label "kst-1 ignore"
			`	"Apologies for invading your personal space, I give." The Kimek quickly walks away.`
			label "anna approaches"
			`	After a few hours, you see Anna enter the lounge and walk toward you. She takes a seat next to you.`
			choice
				`	"How did the interview go?"`
			`	"They told me that due to the review process to make sure it's suitable, I'll have to wait for roughly half a year. Thankfully, they have arranged a small apartment for me to live in. You can go on your own now, but please make sure you come back. If you want to return to human space, please remember to come back and meet me."`
			choice
				`	"Why don't you just go back to human space?"`
					goto stay
				`	"I'll make sure to do that."`
			label thanks
			`	"Thanks again, <first> <last>. I'll forever be grateful to you for helping me. I never thought I'd be successful as an author, but when I met you, you helped me turn my career into something I thought I'd never see."`
			choice
				`	"Just helping out."`
				`	"I'm happy to have helped you."`
			`	"Again, thank you. You will not be needed anymore for the process."`
			choice
				`	(Leave.)`

> 			`	You return to the spaceport the way you came and return to your ship. You are slightly anxious about the outcome of the process and whether Anna's book will even make it through the process and be published, but you have more important things to attend to for the next six months.`

Reviewer 5: ```suggestion
			`	You return to the spaceport the way you came and return to your ship. You are slightly anxious about the outcome of the process and whether Anna's book will even make it through the process to be published, but you have more important things to attend to for the next six months.`
```

		outfit "Jump Drive" -1
		payment 5000000
		fail

mission "Coalition: Struggling Author - Human Media 1"
	minor
	name "Get Kachachee Cultural Information"

> 	description "A group of Coalition citizens have asked you for information on the human race. Rumor says that you can find that information on <stopovers>."

Reviewer 5: ```suggestion
	description "A group of Coalition citizens have asked you for information on the human race. You can find that information on <stopovers>."
```

				decline
			label broke
			`The station has a small museum gift shop that sells a copy of the entire archive on a data card. It only costs 40 credits, but because you have done a horrible job at managing your finances you do not have even that much cash on hand right now.`
			`	Go earn some money, then return here.`
				decline
	on complete
		event "coalition citizens discover fangvin" 20 40
		payment 180000
		conversation
			`You head to the bar where you first met Kachachee. When you arrive, the Kimek greets you, "Thank you. Enlighten me and my friends, this datachip will. A chip of <payment>, as promised, we have." Kachachee places it in your hand.`

event "coalition citizens discover fangvin"

mission "Coalition: Struggling Author - Book Discovered"
	minor
	name "Look for Anna Fangvin"
	description "Get in contact with Anna Fangvin, an author that the Coalition book club has shown interest in."
	source
		government "Coalition"
	destination "Zug"
	to offer
		has "event: coalition citizens discover fangvin"
	on offer
		conversation

> 			`After an uneventful trip to the spaceport, you return to your <model>. Once aboard, you see that you have a message from the group of Coalition citizens that you had provided with a datachip containing a copy of Alexandria's archive:`

Reviewer 5: ```suggestion
			`After an uneventful trip to the spaceport, you return to your <model>. Once aboard, you see a message from the group of Coalition citizens you gave a datachip with a copy of Alexandria's archive:`
```

			``
			`	Dear <first>,`
			`		Author from the datachip, we are interested in. "Anna Fangvin," her name is. Full book, while we could not access, the summary, of your race's actions against the oppressors, captivate us, it did. To converse with her, we wish. A copy of the book, we desire.`
			`	Kachachee`
			``
			choice
				`	(Look for Anna Fangvin.)`
				`	(Maybe later.)`
					defer
				`	(Discard the message.)`
					decline

> 			`	A quick look through your ship's data files reveals nothing. However, checking a copy of the data chip that you gave the Coalition citizens reveals Anna Fangvin to be an author from <destination>. You decide to find her there for an interview.`

Author: Wouldn't it be easier to meet someone than to dig around in hundreds of bookshops on one planet?

			`	As you wait for Anna, you look over your maps of Coalition space to double-check what planet the Coalition book club is on. After a quick look, you find <destination>.`
			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open it to let her inside, lead her to her bunk, and inform her of the destination. You prepare to leave the spaceport.`
				accept
			label decline
			`	Filled with disappointment, Anna says, "Well, I'm afraid I have other things to do, then. Thank you for the stories, at least." She leaves.`
				decline
	on enter
		system
			government "Coalition" "Heliarch"
		conversation
			`As the flames of your jump drive fade away, Anna comes out of her bunk in shock. "What was that? It felt like the whole ship vanished out of our space altogether! Where are we?"`
			choice
				`	"My jump drive would like to welcome you to Coalition space."`
				`	"That was a jump drive."`
				`	"That's what happens every time I go to Coalition space with a jump drive."`
			`	"Wow. I can't believe this is happening! I'm so excited!" Anna looks out the window. "What are those golden ships?"`

> 			`	"They're Heliarch vessels," you reply.`

Reviewer: I'm not sure these lines do us much good. Coalition ships have very little color, so it would be awkward for Anna to call Heliarch ships "golden" when they have so little swizzled surface area; the player could be very confused as they might not ever have even noticed that Coalition/Heliarch ships swizzle *at all*.

I think it would flow better to just continue on with her amazement and curiosity about the JD.

			`	"Well, thank you for bringing me here," Anna says when she finishes. "I didn't sleep too well, so I'll see you later. Goodbye!" She heads back into her cabin, and you settle back into the controls.`
	npc
		government "Heliarch"
		personality heroic opportunistic
		fleet "Heliarch"
	on complete
		set "coalition: struggling author - beyond the hyperdrive"
		conversation
			`Anna gives you the book before you both exit the <ship>. The moment you are out, three Kimek approach you and Anna. The Kimek in the middle speaks, and then a brief silence occurs. After a moment, their translation box breaks the silence.`
			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"`

> 			`	The Coalition is already familiar with you, so you tell them your name while Anna introduces herself. "About what do you write?" asks the Kimek.`

Reviewer 5: They just greeted you by name
```suggestion
			`	The Coalition is already familiar with you, so you wait while Anna introduces herself. "About what do you write?" asks the Kimek.`
```

> 			`	The Coalition is already familiar with you, so you tell them your name while Anna introduces herself. "About what, do you write?" asks the Kimek. "Oh, I write science fiction books," replies Anna.`

Reviewer: Separate characters speaking should do so in separate lines.

			`	The Kimek begin conversing amongst themselves in their native language for a good ten to twenty seconds before they move to face you again.`

> 			`	Then, switching back to your language, the Kimek explains that Anna is welcome to be with you, but reminds you that she will not be allowed to do much unless she registers. "Register?" asks Anna.`

Reviewer 5: ```suggestion
			`	Then, switching the translation box back on, the Kimek explains that Anna is welcome to be with you, but reminds you that she will not be allowed to do much unless she registers. "Register?" asks Anna.`
```

			`	"You have to register yourself with the Heliarchs before you can do much here. I'll take you to the Ring of Friendship, which is where you can complete the registration process." you say.`
			`	"Okay," says Anna, "I'll explore the spaceport a little bit to find some inspiration. Get your ship ready and meet me there."`

mission "Coalition: Book Discovered 2"
	name `"Beyond the Hyperdrive" for Kachachee`
	description `Now that you have a copy of Anna Fangvin's "Beyond the Hyperdrive," give it to Kachachee and their group on <destination>.`
	to offer
		has "coalition: struggling author - beyond the hyperdrive"
	source
		government "Coalition"
		not planet "Ki Patek Ka"
	destination "Ki Patek Ka"
	on offer
		conversation
			`Sorting through messages, you recall that a Kimek named Kachachee and their group wanted a copy of Anna Fangvin's "Beyond the Hyperdrive." You now know her.`
			choice
				`	(Ask Anna for another copy of the book.)`
				`	(Forget about it.)`
					decline

> 			`	You call Anna, and ask her for another copy of "Beyond the Hyperdrive" for a friend named Kachachee. She says, "Sure, I'll send it to you over the Coalition's incredible hyperspace relays. For something as small as a book, there probably won't even need to be a spaceship bringing it! Where do you want it sent?" You reply, "To <planet>." She hangs up, presumably to ship the second copy of "Beyond the Hyperdrive" to <destination>. You should head there to receive and gift it to Kachachee.`

Reviewer 2: This is already implied.
```suggestion
			`	You call Anna, and ask her for another copy of "Beyond the Hyperdrive" for a friend named Kachachee. She says, "Sure, I'll send it to you over the Coalition's incredible hyperspace relays. For something as small as a book, there probably won't even need to be a spaceship bringing it! Where do you want it sent?" You reply, "To <planet>." You should head there to receive and gift it to Kachachee.`
```

> 			`	You call Anna, and ask her for another copy of "Beyond the Hyperdrive" for a friend named Kachachee. She says, "Sure, I'll send it to you over the Coalition's incredible hyperspace relays. For something as small as a book, there probably won't even need to be a spaceship bringing it! Where do you want it sent?" You reply, "To <planet>." She hangs up, presumably to ship the second copy of "Beyond the Hyperdrive" to <destination>. You should head there to receive and gift it to Kachachee.`

Reviewer 2: Why would you go to Ki Patek Ka to give the book to Kachachee if you think that the book will be sent directly to Kachachee anyways?

				accept
	on complete
		payment 7000
		conversation

> 			`After receiving the book - it turns out to have reached there in a normal spacecraft, along with a message from Anna stating her disappointment - from the information counter at a company your translator calls "Hyperspace Relay Incorporated," you get in contact with Kachachee and inform them you have a copy of "Beyond the Hyperdrive." They respond, telling you to meet them at a public square within the spaceport.`

Reviewer 2: ```suggestion
			`After receiving the book - it turns out to have reached there in a normal spacecraft, along with a message from Anna stating her disappointment - you get in contact with Kachachee and inform them you have a copy of "Beyond the Hyperdrive." They respond, telling you to meet them at a public square within the spaceport.`
```

		conversation
			`Going through the spaceport looking for Anna, you finally find her at a restaurant, eating a longcow steak and typing something on her laptop. The Heliarch Saryd from earlier stands beside her, bent-over and gesturing at something on the screen. She sees you and motions for you to take a seat.`
			`	"This meat is so delicious! After months of instant noodles and takeout back home, it finally feels good to eat something that's actually healthy," she exclaims, closing the computer.`
			choice
				`	"How good is it?"`
					goto good
				`	"What did you usually eat back in human space?"`
			`	"My diet wasn't exactly the best. All I had was carbs and takeout food; I never could find time in my schedule to cook at home. I was working hard on 'Beyond the Hyperdrive' at the time. This food is better than anything I could ever find on Zug.`
				goto published
			label good
			`	"It's very good. Probably better than anything I've had back on Zug. Now, I'd like to get my book published here."`
				goto company
			label published
			`	"On another note, I want to get my book published here."`
			label company
			`	"Have you found a publishing company yet?" you ask.`

> 			`	"No," Anna replies. "But this Heliarch has been a huge help in finding a suitable publisher. Without this agent, it would've been nearly impossible for me to communicate with the restaurant staff, since they don't have a translation box."`

Reviewer: Might be good to give the Saryd Heliarch a name, to make it less awkward to keep referring to him as "the Saryd/the Heliarch/the agent".

			choice
				`	"What's the plan now?"`
				`	"Did the Heliarch agent mention anywhere we should go now?"`
			`	The Heliarch straightens up. "Take us to <planet>, you should. Better help there, I can."`
			`	"Does that sound reasonable to you?" Anna asks. When you reply in the affirmative, she says, "We'll be at your spaceship when you are."`
				accept
	on complete
		conversation
			`As you land, Anna says, "Should we see about getting a translation device? I don't want to bother Temporo the whole time I'm here."`
			`	Temporo approaches. "Trust you with a translator, the government cannot. So help you, I must. When found a publisher, you have, accompany you to their offices, I will."`
			`	"We need to find a publishing company then!" says Anna.`
			label questions
			choice
				`	"The Kimek might be a good choice. They have a high population."`
					goto publishingcompany

> 				`	"Perhaps the Saryd. People with human parts are likely to be more interested in human books."`

Reviewer 2: 'human parts'? Why would that make them more receptive to Anna's book?

				goto questions
			label publishingcompany
			action
				clear "struggling author: arachi"
				clear "struggling author: saryd"
			`	"But where can we find a Kimek publishing company? We haven't even translated our book into their language yet!"`
			choice
				`	"We'll find a company."`
				`	"Given the number of books here, there has to be a publishing company somewhere."`
			`	"The faster we can get the book published, the better. Meet me in the spaceport in a few hours; I'll find a good company."`
			`	Before Anna can head off, Temporo rises. "Again accompany you, I will."`

mission "Coalition: Struggling Author 4"
	name "Kimek Publishing Company"
	description "Anna Fangvin found a Kimek publishing company who is willing to get her book published. Bring her to <destination>."
	source "Ki Patek Ka"
	destination "Inmost Blue"
	blocked "You do not have enough bunk space for Anna Fangvin and the Heliarch agent. Come back here when you have two bunks free."
	passengers 2
	to offer
		has "Coalition: Struggling Author 3: done"
	on offer
		conversation

> 			`The spaceport of Ki Patek Ka is vast, and would normally prove a nightmarish place to look for just one person. Thankfully, Anna being one of the few humans within the Coalition makes your search relatively quick. She has chosen to settle in a cafeteria with her laptop. She is typing away at the keyboard, presumably writing a short story to post.`

Reviewer 2: ```suggestion
			`The spaceport of Ki Patek Ka is vast, and would normally prove a nightmarish place to look for just one person. Thankfully, Anna being the only other human here makes your search relatively quick. She has chosen to settle in a cafeteria with her laptop. She is typing away at the keyboard, presumably writing a short story to post.`
```

Author: I'll make the change, but I'm not sure the player could be sure of that.

			`	"<first>! I found a company who's willing to publish me! I sent them a digital copy of my book and they instructed me to visit their office on <planet>," explains Anna, once again enthusiastic.`
			choice
				`	"Great!"`
					goto go
				`	"Let's go!"`
					goto go
				`	"What's the publishing company called?"`
			`	"They're called 'Priyepichia Publishing.'" Anna shows you beautiful pictures of a clean white building perched on a mountainside, high above gentle waves. On the roof stands a landing pad painted with a blue dolphin.`
			label go

> 			`	Anna heads to your ship, and you plot a course to <destination>. Just before you launch, the Heliarch hops in.`

Reviewer: The Heliarch just hopping in last minute is odd.

I think we could:

- Have it so that he's again helping Anna when you find her, and follows you two to the ship, or
- Have it so that he had gone to look for info elsewhere, and when you and Anna are returning to your ship  you find him waiting there, and catch up briefly just before boarding and launching.

				accept
	on complete
		event "beyond the hyperdrive is published" 165 197
		conversation
			`Landing on <planet>, Anna is excitedly holding a copy of 'Beyond the Hyperdrive.' She hastily opens the hatch as soon as it unlocks.`
			`	Anna stops, waiting for you to catch up. "In the message they sent me, they included instructions on how to reach their offices," she explains. "Let's go!"`
			`	Several long streets later, Anna is completely lost. "This isn't where the instructions were supposed to lead at all," she says, turning to Temporo, who has been following you the whole time. "Do you know how to get there?"`

> 			`	The Heliarch takes the lead, and within minutes you reach a train station. Temporo leads you to platform 61A. When the train arrives, you board it and travel to a neighborhood where a large white building dominates the skyline.`

Reviewer 2: Is that detail really necessary?
```suggestion
			`	The Heliarch takes the lead, and within minutes you reach a train station. Once the train arrives, you travel to a neighborhood where a large white building dominates the skyline.`
```

> 			`	Once the train stops, you get off onto the platform and exit the station. Circling around to the publisher's office, you find the entrance bustling with Kimek, along with the occasional Saryd and Arach coming and going. Anna looks for a front desk and directs you to a Kimek who appears to be a receptionist.`

Reviewer 2: ```suggestion
			`	Circling around to the publisher's office, you find the entrance bustling with Kimek, along with the occasional Saryd and Arach coming and going. Anna looks for a front desk and directs you to a Kimek who appears to be a receptionist.`
```

> 			`	Once the train stops, you get off onto the platform and exit the station. Circling around to the publisher's office, you find the entrance bustling with Kimek, along with the occasional Saryd and Arach coming and going. Anna looks for a front desk and directs you to a Kimek who looks appears to be a receptionist.`

Reviewer: The Saryd Heliarch isn't really acknowledged here. I would think that he'd be more capable of guiding you three through the city and to the publishing company. Adding mention of him being present later in the interview for the book would also be good.

			choice
				`	"Hello, do you happen to know where Priyepichia Publishing is?"`
				`	"We're looking for a publishing company called Priyepichia Publishing. Could you provide directions?"`
			`	"On the 16th floor, Priyepichia Publishing is. Over there, the elevators are," says the receptionist's translator.`
			`	You turn to where the Kimek points you and see six large elevators, larger than what you are familiar with in human space. Temporo leads you to the least crowded one.`

> 			`	The three of you enter the elevator. It makes stops at each floor of the building, more and more Kimek reach their destinations, and the elevator gets more and more quiet. When you reach the 16th floor, the elevator is only half full. The door opens to reveal a Kimek, wearing an ornate jeweled necklace as well as a translation device.`

Reviewer 2: ```suggestion
			`	The three of you enter the elevator. It makes stops at each floor of the building, and the elevator gets quieter as more Kimek reach their destinations. When you reach the 16th floor, the elevator is only half full. The door opens to reveal a Kimek, wearing an ornate jeweled necklace as well as a translation device.`
```

> 			`	The three of you enter the elevator. It makes stops at each floor of the building, more and more Kimek reach their destinations, and the elevator gets more and more quiet. When the door opens on the 16th floor, the elevator is only half full. Before you stands a Kimek who boarded at the eighth floor, wearing an ornate jeweled necklace as well as a translation device.`

Reviewer 5: Kind of odd that the publishing company would have their greeter get on the elevator with you on a random floor and not do anything. Probably makes more sense for the greeter to be waiting on the publisher's floor instead.
```suggestion
			`	The three of you enter the elevator. It makes stops at each floor of the building, more and more Kimek reach their destinations, and the elevator gets more and more quiet. When you reach the 16th floor, the elevator is only half full. The door opens to reveal a Kimek, wearing an ornate jeweled necklace as well as a translation device.`
```

			choice
				`	"Hello?"`
				`	(Exit the elevator.)`
			`	"Greetings, humans. Follow me, you must, so discuss your book, we can," the Kimek says through its translation box.`
			`	At the direction of the Kimek, you and Anna walk as directed until you arrive at a room with glass walls. Along the way, you pass a lounge, a conference area where multiple Kimek are discussing something at a table with a group of Heliarchs, and a large open workspace where individual Kimek sit at their own computers.`

> 			`	After a few minutes of walking, you reach an office where another Kimek sits with a team of five Heliarchs - two from each species except for the Saryds. Temporo takes a seat with them.`

Author: Temporo was supposed to be the sixth Heliarch. Would this clarify it?
```suggestion
			`	After a few minutes of walking, you reach an office where another Kimek sits with a team of Heliarchs. Temporo takes a seat with them, making six - two from each species.`
```

> 			`	After a few minutes of walking, you reach an office where another Kimek sits with a team of five Heliarchs - two from each species except for the Saryds. Temporo takes a seat with them.`

Reviewer 3: Yes

			`	The Kimek guiding you gestures toward the office and heads over to speak with the group.`
			`	As you enter, Anna puts down the copy of her book on the table, and you each take a seat. The Kimek that led you here briefly speaks to the other non-Heliarch Kimek sitting at the interview table before leaving.`
			`	The sitting Kimek's translation device begins speaking.`
			`	"Greetings. Ikiyachee, my name is, and head editor of Priyepichia Publishing, I am. To see an alien want to publish a book, interesting and unexpected it was," the Kimek says.`
			`	"Hello, I am Anna Fangvin, and this is my friend <first> <last>, who has been very helpful in bringing me here," Anna says, somewhat flustered.`

> 			`	"From the introduction and the first few pages, interesting your book sounded, but run it past the Heliarch reviewers, we must. So, <first>, long and intensive, this interview will be. As very unique, to meet a human like her is, and to adapt her book to our culture, a long and drawn-out process it will be."`

Reviewer 3: I just noticed how Ikiyachee uses the terms "human" and "alien" interchangeably here to refer to Anna.

Author: Well, they're essentially synonymous. Other than Coalition species, humans are the only aliens they've met.

			choice
				`	"Do I have to stay with Anna here for half a year?"`
					goto "half a year"
				`	(Say nothing.)`
			label "rest of interview"
			`	As the hours pass, Anna's enthusiastic tone remains strong and constant. The Kimek then takes the book, puts it in a nearby filing cabinet and thanks the two of you.`
			`	You leave the interview room, while the Heliarchs stay, presumably to continue speaking with the Kimek. As the two of you walk out, Anna begins to speak.`
				goto thanks
			label "half a year"
			`	"Required to stay with Anna, you are not, but if you wish to, stay with her you may. Return to human space, you are free to do," says the Kimek.`
				goto "rest of interview"
			label "lounge"
			`	You leave the room and walk to the lounge. There are several tables where many Kimek are chatting in their native language, while a few Saryds and Arachi are visibly relaxed. You sit at an empty table to pass the time, occasionally going to the water dispenser to hydrate yourself.`

> 			`	After thirty minutes, a Kimek approaches you. Their translation device speaks. "An alien in our space, you are? Most strange, this meeting truly is. Coalition member the like of you, have I never encountered. Saryd or Arachi, are you?"`

Reviewer 2: Does this guy not know what a human is? I'm pretty sure that your average Coalition citizen is at least aware humans exist, and wouldn't mistake them for a Saryd or Arachi. 
```suggestion
			`	After thirty minutes, a Kimek approaches you. Their translation device speaks. "An alien in our space, you are? Most strange, this meeting truly is. A Coalition member the likes of you, have I never encountered. Saryd or Arachi, are you?"`
```

			`	The Kimek walks away after the two of you exchange farewells, leaving you to your own devices.`
				goto "anna approaches"
			label "kst-1 ignore"
			`	"Apologies for invading your personal space, I give." The Kimek quickly walks away.`
			label "anna approaches"
			`	After a few hours, you see Anna enter the lounge and walk toward you. She takes a seat next to you.`
			choice
				`	"How did the interview go?"`
			`	"They told me that due to the review process to make sure it's suitable, I'll have to wait for roughly half a year. Thankfully, they have arranged a small apartment for me to live in. You can go on your own now, but please make sure you come back. If you want to return to human space, please remember to come back and meet me."`
			choice
				`	"Why don't you just go back to human space?"`
					goto stay
				`	"I'll make sure to do that."`
			label thanks
			`	"Thanks again, <first> <last>. I'll forever be grateful to you for helping me. I never thought I'd be successful as an author, but when I met you, you helped me turn my career into something I thought I'd never see."`
			choice
				`	"Just helping out."`
				`	"I'm happy to have helped you."`

> 			`	"Again, thank you. You will not be needed anymore for the process."`

Reviewer 2: I know that this probably isn't intentional, but this sounds kind of rude for her to say?

> 			`	You return to the spaceport the way you came and return to your ship. You have more important things to attend to for the next six months than wonder whether Anna's book will make it through the process to be published.`

Reviewer 2: ```suggestion
			`	You return to the spaceport the way you came and return to your ship.`
```

				accept
	on complete
		payment 750000
		conversation "anna invites you to book signing"

conversation "anna invites you to book signing"
	action
		log "People" "Anna Fangvin" `Anna was a struggling science fiction author in human space until she found success publishing her novel 'Beyond the Hyperdrive' in the Coalition. Her book is a bestseller in the Kimek region.`
		log "Helped somebody named Anna Fangvin get her book published in Coalition space. Apparently, it's a bestseller now. The book was 'Beyond the Hyperdrive', something about humans stealing jump drives from the Quarg."
		event "beyond the hyperdrive payment" 40
		set "beyond the hyperdrive news can show now"
		outfit "Beyond the Hyperdrive" 1
	`The moment you land, Anna arrives, very happy to see you. She grabs your shoulders with both arms and in an excited tone explains that her book has been approved for publication.`
	`	"They're publishing 'Beyond the Hyperdrive' today, and there's a full release event where I'll be signing copies! Thanks for everything you've done, <first>! Do you want to join me during the book signing?"`

> 	choice

Reviewer 2: No option to decline?

			`	She only seems to notice you when you also sit down beside her. "Oh, hello again, <first>!" she says, snapping back into focus. "Is the ship ready?"`
			`	Hearing an affirmative answer, Anna stands up from her chair and begins following you to the <ship>. She settles into her bunk as you prepare for the trip to the <destination>.`
				accept
	on enter "Quaru"
		conversation
			`As you enter <system>, Anna walks out of her bunk and finds a spot on the side of the bridge, peering into the empty void. "What a massive ringworld!" she exclaims.`
			choice
				`	"It's enormous. As you know, they stole it from the Quarg roughly seven thousand years ago."`
				`	(Say nothing.)`
			`	"This is going to be a great source of inspiration for my next novel," says Anna, still in awe. She pulls out a small notebook and begins busily writing in it.`
	on complete
		conversation
			`Anna is amazed as you touch down on the ringworld. As the two of you open the hatch, you see a Heliarch agent in front of you.`

> 			`	"Greetings, humans. For you, what can I do today?" asks the Heliarch. You respond that you would like to register Anna with the Heliarchy. After reaching the registration office, Anna enters it, coming out after fifteen minutes. "They finally got all the paperwork done. I'm off to the spaceport to get some food. Meet me there to discuss our next move." The Heliarch follows her to the cafeteria area.`

Reviewer 5: ```suggestion
			`	"Greetings, humans. For you, what can I do today?" asks the Heliarch. You respond that you would like to register Anna with the Heliarchy. Upon reaching the registration office, Anna enters, and fifteen minutes later exits alongside another Heliarch, this time a crowned Saryd. "They finally got all the paperwork done. We're off to the spaceport to get some food. Meet us there to discuss our next move."`
```

mission "Coalition: Struggling Author 3"
	name "Back to <planet>"
	description "Now that Anna Fangvin has completed the registration process, take her and the Heliarch agent to get a translation device on <destination>."
	source "Ring of Friendship"
	destination "Ki Patek Ka"
	passengers 2
	blocked "You do not have enough space for Anna and the Heliarch agent. Return here when you have two bunks free."
	to offer
		has "Coalition: Struggling Author 2: done"
	on offer
		conversation

> 			`Going through the spaceport looking for Anna, you finally find her at a restaurant, eating a longcow steak and typing something on her laptop. Beside her sits a Heliarch Saryd, who points at something on the screen. She sees you and motions for you to take a seat.`

Reviewer 5: Horse (and I presume Centaurs/Saryds?) don't have the anatomy to sit naturally without great discomfort and possible damage - they're much more comfortable standing. Tweaked for the Heliarch to be standing instead.
```suggestion
			`Going through the spaceport looking for Anna, you finally find her at a restaurant, eating a longcow steak and typing something on her laptop. The Heliarch Saryd from earlier stands beside her, bent-over and gesturing at something on the screen. She sees you and motions for you to take a seat.`
```

			`	"This meat is so delicious! After months of instant noodles and takeout back home, it finally feels good to eat something that's actually healthy," she exclaims, closing the computer.`
			choice
				`	"How good is it?"`
					goto good
				`	"What did you usually eat back in human space?"`
			`	"My diet wasn't exactly the best. All I had was carbs and takeout food; I never could find time in my schedule to cook at home. I was working hard on 'Beyond the Hyperdrive' at the time. This food is better than anything I could ever find on Zug.`
				goto published
			label good
			`	"It's very good. Probably better than anything I've had back on Zug. Now, I'd like to get my book published here."`
				goto company
			label published
			`	"On another note, I want to get my book published here."`
			label company
			`	"Have you found a publishing company yet?" you ask.`

> 			`	"No," Anna replies. "But this Heliarch has been very helpful in the search for an adequate publishing company. Had it not been for the agent, it would have been extremely difficult for me to communicate with the restaurant staff, as they do not have a translation box."`

Reviewer 5: ```suggestion
			`	"No," Anna replies. "But this Heliarch has been a huge help in finding a suitable publisher. Without this agent, it would've been nearly impossible for me to communicate with the restaurant staff, since they don't have a translation box."`
```

			choice
				`	"You'll need to have the Heliarchs help translate for you."`

> 				`	"Will the Heliarch agent help?"`

Reviewer 5: I don't quite think this makes sense anymore, since Anna just said how helpful this agent is, and how helpful they've been translating, so why would you ask about those same things? Maybe now they say that X planet has a lot of publishing companies? 

Might need further tweaks though. This agent already with you would know that they don't hand out translator boxes to just anyone, but also she doesn't need one since the agent is already translating for her.

	name `"Beyond the Hyperdrive" for Kachachee`
	description `Now that you have a copy of Anna Fangvin's "Beyond the Hyperdrive," give it to Kachachee and their group on <destination>.`
	to offer
		has "coalition: struggling author - beyond the hyperdrive"
	source
		government "Coalition"
		not planet "Ki Patek Ka"
	destination "Ki Patek Ka"
	on offer
		conversation
			`Sorting through messages, you recall that a Kimek named Kachachee and their group wanted a copy of Anna Fangvin's "Beyond the Hyperdrive." You now know her.`
			choice
				`	(Ask Anna for another copy of the book.)`
				`	(Forget about it.)`
					decline
			`	You call Anna, and ask her for another copy of "Beyond the Hyperdrive" for a friend named Kachachee. She says, "Sure, I'll send it to you over the Coalition's incredible hyperspace relays. For something as small as a book, there probably won't even need to be a spaceship bringing it! Where do you want it sent?" You reply, "To <planet>." She hangs up, presumably to ship the second copy of "Beyond the Hyperdrive" to <destination>. You should head there to receive and gift it to Kachachee.`
				accept
	on complete
		payment 7000
		conversation

> 			`After receiving the book from the information counter at a company - it turns out to have reached there in a normal spacecraft, to Anna's disappointment - your translator calls "Hyperspace Relay Incorporated," you get in contact with Kachachee and inform them you have a copy of "Beyond the Hyperdrive." They respond, telling you to meet them at a public square within the spaceport.`

Reviewer 5: Anna may not be with you.
```suggestion
			`After receiving the book from the information counter at a company - it turns out to have reached there in a normal spacecraft, your translator calls it "Hyperspace Relay Incorporated," you get in contact with Kachachee and inform them you have a copy of "Beyond the Hyperdrive." They respond, telling you to meet them at a public square within the spaceport.`
```

Author: ```suggestion
			`After receiving the book - it turns out to have reached there in a normal spacecraft, along with a message from Anna stating her disappointment - from the information counter at a company your translator calls "Hyperspace Relay Incorporated," you get in contact with Kachachee and inform them you have a copy of "Beyond the Hyperdrive." They respond, telling you to meet them at a public square within the spaceport.`
```

			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open the hatch to let her inside, lead her to her bunk, and inform her of the destination. You prepare to leave the spaceport.`
				accept
			label decline
			`	Filled with disappointment, Anna says, "Well, I'm afraid I have other things to do, then. Thank you for the stories, at least." She leaves.`
				decline
	on enter
		system
			government "Coalition" "Heliarch"
		conversation
			`As the flames of your jump drive fade away, Anna comes out of her bunk in shock. "What was that? It felt like the whole ship vanished out of our space altogether! Where are we?"`
			choice
				`	"Welcome to Coalition space, Anna."`
				`	"That was a jump drive."`
				`	"That's what happens every time I go to Coalition space."`
			`	"Wow. I can't believe this is happening! I'm so excited!" Anna looks out the window. "What are those golden ships?"`
			`	"They're Heliarch vessels," you reply.`
			`	"Can I take a look at your Jump Drive? I'm interested to see how it works," Anna says. You take her to the engine room of <ship>, where she begins taking notes and pictures of the drive.`

> 			`	"Well, thank you for bringing me here," Anna says when she finishes. "I didn't sleep too well, so I'll see you later. Goodbye!" She heads back into her cabin, and you settle down to the controls.`

Reviewer 5: ```suggestion
			`	"Well, thank you for bringing me here," Anna says when she finishes. "I didn't sleep too well, so I'll see you later. Goodbye!" She heads back into her cabin, and you settle back into the controls.`
```

			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"`
			`	The Coalition is already familiar with you, so you tell them your name while Anna introduces herself. "About what, do you write?" asks the Kimek. "Oh, I write science fiction books," replies Anna.`
			`	The Kimek begin conversing amongst themselves in their native language for a good ten to twenty seconds before they move to face you again.`
			`	Then, switching back to your language, the Kimek explains that Anna is welcome to be with you, but reminds you that she will not be allowed to do much unless she registers. "Register?" asks Anna.`
			`	"You have to register yourself with the Heliarchs before you can do much here. I'll take you to the Ring of Friendship, which is where you can complete the registration process." you say.`
			`	"Okay," says Anna, "I'll explore the spaceport a little bit to find some inspiration. Get your ship ready and meet me there."`

mission "Coalition: Struggling Author 2"

> 	name "Help Anna Register"

Reviewer 5: This should still fit
```suggestion
	name "Help Anna Register with the Heliarchs"
```

Author: ```suggestion
	name "Help Anna register with the Heliarchs"
```

			choice
				`	"It's enormous. As you know, they stole it from the Quarg roughly seven thousand years ago."`
				`	(Say nothing.)`
			`	"This is going to be a great source of inspiration for my next novel," says Anna, still in awe. She pulls out a small notebook and begins busily writing in it.`
	on complete
		conversation
			`Anna is amazed as you touch down on the ringworld. As the two of you open the hatch, you see a Heliarch agent in front of you.`
			`	"Greetings, humans. For you, what can I do today?" asks the Heliarch. You respond that you would like to register Anna with the Heliarchy. After reaching the registration office, Anna enters it, coming out after fifteen minutes. "They finally got all the paperwork done. I'm off to the spaceport to get some food. Meet me there to discuss our next move."`

mission "Coalition: Struggling Author 3"
	name "Back to <planet>"
	description "Now that Anna Fangvin has completed the registration process, help her get a translation device on <destination>."
	source "Ring of Friendship"
	destination "Ki Patek Ka"
	passengers 1
	to offer
		has "Coalition: Struggling Author 2: done"
	on offer
		conversation

> 			`Going through the spaceport looking for Anna, you finally find her at a restaurant, eating food made for Saryds. She's typing away furiously at her laptop with a plate of vegetables next to her. She sees you and motions for you to take a seat.`

Reviewer 5: "food made for Saryds" should be more specific, is it a longcow steak, multi-colored fruit, or something else entirely? I don't know if looking at a plate of food would be obvious what species it's made for, nor obvious what the food is if it's made for Saryds.

One idea using longcow steaks:
```suggestion
			`Going through the spaceport looking for Anna, you finally find her at a restaurant, eating a Longcow steak. She's typing away furiously at her laptop with a plate of vegetables next to her. She sees you and motions for you to take a seat.`
```

Author: ```suggestion
			`Going through the spaceport looking for Anna, you finally find her at a restaurant, eating a longcow steak. She's typing away furiously at her laptop with a plate of vegetables next to her. She sees you and motions for you to take a seat.`
```

			choice
				`	"You'll need to have a Heliarch agent help translate for you."`
				`	"Ask Heliarch agents to help."`
			`	"Great! So, where will we get permission to do so?"`
			`	A Heliarch at a nearby table stands and approaches. "Get an escort on <planet>, you should. Duties assigned to us, we have here."`
			`	"Does that sound fair to you?" Anna asks. When you reply in the affirmative, she says, "I'll be at your spaceship when you are."`
				accept
	on complete
		conversation
			`You arrive at the spaceport. Anna leaves her bunk and waits for you to open the hatch. You quickly lead her to a Heliarch office. You walk to the counter, and begin conversing with the clerk.`
			`	"Greetings, human. To be of assistance to you, how can I?" the clerk says.`
			`	Anna speaks up. "I'm Anna Fangvin, and I'd like to publish a book here in Coalition space. My friend <first> <last> tells me I will need a translation box to converse with the publishing company. Could you provide one?"`
			`	"A translator, the human needs?" asks the clerk.`
			choice

> 				`	"Yes."`

Reviewer 5: Maybe add another option here?
```suggestion
				`	"Yes."`
				`	"I think so."`
```

			`	"Trust you with a translator, we cannot. And help you, I cannot, so a Heliarch, I must call," says the clerk, then proceeding to call for a Heliarch in its native language.`
			`	After a few minutes, another Kimek wearing the telltale golden circlet of a Heliarch approaches you and directs you into a room.`

> 			`	"Told me about your intentions, the clerk already has, and understand it, I do. Accompany you to the publishers, me and my agents will, once decided you have upon a company." The Arachi sits down.`

Reviewer 5: Trying to tweak for more yoda-like speech
```suggestion
			`	"Your intentions, the clerk has already told me about, and understand it, I do. To the publishers, my agents and I will accompany you, once decided you have upon a company." The Kimek sits down.`
```

				goto questions
			label saryd
			action
				set "struggling author: saryd"
			`	"I thought the same at first, but from what I've seen the Saryds seem pretty reserved, don't you think? Maybe another species would be better."`
				goto questions
			label publishingcompany
			action
				clear "struggling author: arachi"
				clear "struggling author: saryd"
			`	"But where can we find a Kimek publishing company? We haven't even translated our book into their language yet!"`
			choice
				`	"We'll find a company."`
				`	"Given the number of books here, there has to be a publishing company somewhere."`
			`	"The faster we can get the book published, the better. Meet me in the spaceport in a few hours; I'll find a good company."`
			`	Before Anna can head off, the Heliarch rises. "Accompany you, I will. Suitable for two aliens to roam around our space unattended, it is not."`

mission "Coalition: Struggling Author 4"
	name "Kimek Publishing Company"
	description "Anna found a Kimek publishing company who is willing to take in Anna Fangvin to get her book published. Bring Anna Fangvin to <destination>."
	source "Ki Patek Ka"
	destination "Inmost Blue"

> 	blocked "You do not have enough bunk space for the Heliarch agent and Anna Fangvin. Come back here when you have two bunks free."

Reviewer 3: ```suggestion
	blocked "You do not have enough bunk space for Anna Fangvin and the Heliarch agent. Come back here when you have two bunks free."
```

Probably should put Anna's name first, as she is more relevant to the storyline than the Heliarch agent.

			`	"Same. 'Akacharra', my name is. To our fine neighborhood, for what purpose have you come here?"`
			`	"My friend Anna Fangvin is publishing a book here," you say.`
				goto "kst-1 anna"
			label "kst-1 not only alien"
			`	"If more of you, here there are, then who are they?"`
			`	"It's just two; my friend Anna Fangvin and I. She's publishing a book in your space," you reply.`
				goto "kst-1 anna"
			label "kst-1 endonym"
			`	"'Humans', you are? Know that, I never did. For correcting me and educating me on your fellow kind, I am thankful," says the Kimek. "So, the other human, who is she?"`
			`	"Oh, that's my friend Anna Fangvin. She's here to publish her book in Coalition space," you answer.`
			label "kst-1 anna"
			`	"Anna Fangvin, her name is? An interesting name, it is, and her book, just as interesting it will be when it is released. But know the title of the book, I do not, so if tell me you will, appreciate it I will."`
			choice
				`	"It is called 'Beyond the Hyperdrive.' It could very well be years before release, but I'm glad you're taking interest."`
				`	"It is called 'Beyond the Hyperdrive.' I think it's likely to be one of the best books of all time."`

> 			`	"So, 'Beyond the Hyperdrive,' it is, you say? Thankful, I am, and on its release, buy the book I will remember to."`

Reviewer 5: ```suggestion
			`	"So, 'Beyond the Hyperdrive,' it is, you say? Thankful, I am. On its release, buy the book, I will remember to."`
```

				goto "anna approaches"
			label "kst-1 ignore"
			`	"Apologies for invading your personal space, I give." The Kimek quickly walks away.`
			label "anna approaches"
			`	After a few hours, you see Anna enter the lounge and walk toward you. She takes a seat next to you.`
			choice
				`	"How did the interview go?"`
			`	"They told me that due to the review process to make sure it's suitable, I'll have to wait for roughly half a year. Thankfully, they have arranged a small apartment for me to live in. You can go on your own now, but please make sure you come back. If you want to return to human space, please remember to come back and meet me."`
			choice
				`	"Why don't you just go back to human space?"`
					goto stay
				`	"I'll make sure to do that."`
			label thanks
			`	"Thanks again, <first> <last>. I'll forever be grateful to you for helping me. I never thought I'd be successful as an author, but when I met you, you helped me turn my career into something I thought I'd never see."`
			choice
				`	"Just helping out."`
				`	"I'm happy to have helped you."`
			`	"Again, thank you. You will not be needed anymore for the process."`

> 			choice
> 				`	(Leave.)`

Reviewer 5: Don't need this
```suggestion
```

	name "Kimek Publishing Company"
	description "Anna Fangvin found a Kimek publishing company who is willing to get her book published. Bring her to <destination>."
	source "Ki Patek Ka"
	destination "Inmost Blue"
	blocked "You do not have enough bunk space for Anna Fangvin and the Heliarch agent. Come back here when you have two bunks free."
	passengers 2
	to offer
		has "Coalition: Struggling Author 3: done"
	on offer
		conversation
			`The spaceport of Ki Patek Ka is vast, and would normally prove a nightmarish place to look for just one person. Thankfully, Anna being the only other human within the Coalition makes your search relatively quick. She has chosen to settle in a cafeteria with her laptop. She is typing away at the keyboard, presumably writing a short story to post.`
			`	"<first>! I found a company who's willing to publish me! I sent them a digital copy of my book and they instructed me to visit their office on <planet>," explains Anna, once again enthusiastic. At a table behind her you notice the figure of Temporo, speaking to a waiter.`
			choice
				`	"Great!"`
					goto go
				`	"Let's go!"`
					goto go
				`	"What's the publishing company called?"`

> 			`	"They're called 'Priyepichia Publishing.'" Anna shows you beautiful pictures of a clean white building perched on a mountainside, high above gentle waves. On the roof stands a landing pad painted with a blue dolphin.`

Author: Apparently.
@Arachi-Lover do you have input on that?

> 			`	"They're called 'Priyepichia Publishing.'" Anna shows you beautiful pictures of a clean white building perched on a mountainside, high above gentle waves. On the roof stands a landing pad painted with a blue dolphin.`

Reviewer: O

Should probably either remove that or make it a clearly-not-from-Earth fish.

> 			`	"They're called 'Priyepichia Publishing.'" Anna shows you beautiful pictures of a clean white building perched on a mountainside, high above gentle waves. On the roof stands a landing pad painted with a blue dolphin.`

Reviewer 5: Couple of options, easiest is probably "fish-like creature":
```suggestion
			`	"They're called 'Priyepichia Publishing.'" Anna shows you beautiful pictures of a clean white building perched on a mountainside, high above gentle waves. On the roof stands a landing pad painted with a blue fish-like creature.`
```

Or a bit more flowery and resembling more of a [water tiger](https://en.wikipedia.org/wiki/Dytiscidae), so that the Kimek company is leaning more into a beetle theme:
```suggestion
			`	"They're called 'Priyepichia Publishing.'" Anna shows you beautiful pictures of a clean white building perched on a mountainside, high above gentle waves. On the roof stands a landing pad painted with a striped, sleek beetle-like creature whose hind legs resemble broad paddles.`
```

	on decline
		clear "beyond the hyperdrive - asked for book copy"
	on enter
		system
			government "Coalition" "Heliarch"
		conversation
			`As the flames of your jump drive fade away, Anna comes out of her bunk in shock. "What was that? It felt like the whole ship vanished out of our space altogether! Where are we?"`
			choice
				`	"My jump drive would like to welcome you to Coalition space."`
				`	"That was a jump drive."`
				`	"That's what happens every time I go to Coalition space with a jump drive."`
			`	"Wow. I can't believe this is happening! I'm so excited!" Anna looks out the window, then turns back.`
			`	"Can I take a look at your jump drive? I'm interested to see how it works," Anna says. You take her to the engine room of <ship>, where she begins taking notes and pictures of the drive.`
			`	"Well, thank you for bringing me here," Anna says when she finishes. "I didn't sleep too well, so I'll see you later. Goodbye!" She heads back into her cabin, and you settle back into the controls.`
	npc
		government "Heliarch"
		personality heroic opportunistic
		fleet "Heliarch"
	on complete
		set "coalition: struggling author - beyond the hyperdrive"
		conversation

> 			`Anna gives you the book before you both exit the <ship>. The moment you are out, three Kimek approach you and Anna.`

Author: Presumably, they are Heliarchs, given that they remind Anna to register. ~~And the Heliarchy knows everything.~~

> 			`Anna gives you the book before you both exit the <ship>. The moment you are out, three Kimek approach you and Anna.`

Reviewer 2: If they're Heliarchs, why isn't that mentioned? Right now it just sounds like they're some random Kimek.

> 			`Anna gives you the book before you both exit the <ship>. The moment you are out, three Kimek approach you and Anna.`

Author: ```suggestion
			`Anna gives you the book before you both exit the <ship>. The moment you are out, three Kimek with golden circlets on their heads approach you and Anna.`
```

				goto "rest of interview"
			label "lounge"
			`	You leave the room and walk to the lounge. There are several tables where many Kimek are chatting in their native language, while a few Saryds and Arachi are visibly relaxed. You sit at an empty table to pass the time, occasionally going to the water dispenser to hydrate yourself.`
			`	After thirty minutes, a Kimek approaches you. Their translation device speaks. "An alien in our space, you are? Most strange, this meeting truly is. A Coalition member the likes of you, I have never encountered. Or human, are you?"`
			# kst-1 is short for [K]imek [s]mall [t]alk 1
			choice
				`	"I'd prefer to be left alone right now."`
					goto "kst-1 ignore"
				`	"Yes, I am the human <first> <last>. Glad to meet you."`
				`	"Well, I'm not the only alien."`
					goto "kst-1 not only alien"
				`	"We call ourselves humans, not aliens."`
					goto "kst-1 endonym"
			label "kst-1 glad to meet you"
			`	"Same. 'Akacharra', my name is. To our fine neighborhood, for what purpose have you come here?"`
			`	"My friend Anna Fangvin is publishing a book here," you say.`
				goto "kst-1 anna"
			label "kst-1 not only alien"

> 			`	"If more of you, here there are, then who are they?"`

Author: What change did you want?

> 			`	"If more of you, here there are, then who are they?"`

Reviewer 2: Sorry, don't know why I made this comment.

			choice
				`	"Why don't you just go back to human space?"`
					goto stay
				`	"I'll make sure to do that."`
			label thanks
			`	"Thanks again, <first> <last>. I'll forever be grateful to you for helping me. I never thought I'd be successful as an author, but when I met you, you helped me turn my career into something I thought I'd never see."`
			choice
				`	"Just helping out."`
				`	"I'm happy to have helped you."`
			`	"Again, thank you for all your help."`
			`	You return to the spaceport the way you came and return to your ship.`
				decline
			label stay
			`	"I need to stay here so I can answer their questions during the review process. Don't worry about me."`
			choice
				`	"Got it. Good luck publishing your book."`
					goto thanks

event "beyond the hyperdrive is published"

mission "Coalition: Beyond the Hyperdrive Fail Book Discovered 2"
	landing
	invisible
	to offer
		has "event: beyond the hyperdrive is published"
	on offer
		fail "Coalition: Book Discovered 2"
		fail

mission "Coalition: Struggling Author 5A"
	landing
	name "Visit Anna"
	description "Visit the author Anna Fangvin on <destination>."
	source

> 		not near "Quaru" 0 100

Reviewer 2: Shouldn't this just offer anywhere that's not in Coalition space?
```suggestion
		not government "Coalition"
```

	destination "Inmost Blue"
	to offer
		has "event: beyond the hyperdrive is published"
		not "Coalition: Struggling Author 5B: offered"
	on offer
		conversation
			`Checking your computer feed for the latest galactic news and trade prices, you get a reminder.`
			`	You read it, and recall that you had helped an author named Anna Fangvin get her book, "Beyond the Hyperdrive," published about half a year ago in Coalition space. You should return to <destination> and check on her.`
				accept
	on complete
		payment 750000
		conversation "anna invites you to book signing"

mission "Coalition: Struggling Author 5B"
	landing
	name "Visit Anna"
	description "Visit the author Anna Fangvin on <destination>."
	source

> 		near "4 Winter Rising" 1 100

Reviewer 2: Same as above.
```suggestion
		government "Coalition"
```

				`	"I see. I will bring you all the knowledge you want."`
				`	"I'm not interested. Sorry."`
					decline
			`	"Thank you. Enlighten us and the rest of the Coalition, your actions will. A safe trip, we hope you have." They give you instructions on where to find them.`
			branch "don't have"
				not "human cultural data"
			`	You recall that your ship has a datachip containing a copy of Alexandria's archives, which may be what the group wants. You walk to your ship, make a copy of the datachip, and return to the group.`
			`	Kachachee then approaches you as you hand them the datachip. "Very speedy, your return was."`
			choice
				`	"Happy to help."`
				`	"I hope you enjoy whatever my race has to offer for you."`
			action
				event "coalition citizens discover fangvin" 14 27
				payment 180000
			`	"Thank you. Enlighten me and my friends, this datachip will. And as promised, a chip of <payment>, we have." Kachachee places it in your hand.`
				decline
			label "group question"
			action
				set "coalition: asked about kachachee"
			`	"A group who likes to read books together, we are. Science fiction, I believe the term is, especially."`
				goto decision
			label "don't have"

> 			`	On <stopovers> there is a library with complete copies of all human history. You return to your ship and prepare for the journey there to acquire one.`

Reviewer 2: In other missions where you're asked for the Alexandria archives, you don't know where the archives are without being told.

Reviewer 2: I think this is a bit more specific.
```suggestion
			`	A quick search of the Worlds Wide Web reveals that the library of <stopovers> has data chips containing a complete database of all human culture and history. You return to your ship and prepare for the journey there to acquire one.`
```

			choice
				`	(Tell her about the Coalition.)`
				`	"No actually, I'm afraid I don't know anyone who could help you."`
					decline
			label coalition
			`	You begin explaining the Coalition to Anna, going in-depth about their culture and their long history of opposing the Quarg, seeing as Anna has less than positive opinions of them. At first Anna does not believe you, but you reassure her it's real. You also mention the population of the Coalition and the novelty of an "alien" author coming into the area.`
			`	"Will they really like my books?"`
			choice
				`	"They will, hopefully."`
				`	"Well, it's better to try than to just give up."`
			`	"These stories might just inspire my next novel. But, they wouldn't speak our language. How are they going to read my books?" You reply, "They have a translation device."`
			`	"That's reassuring... but, how am I even going to get there?" she queries.`
			choice
				`	"I'll take you there."`
				`	"I don't think you can."`
					goto decline

> 			`	"Do I have to pay anything? I can't afford off-world transport at all, let alone to a completely different segment of the galaxy! Actually, I know - I'll give you a copy of Beyond the Hyperdrive! Thank you so much! Just wait for a while. I'll go over to my house and bring what I'll need for the trip, and then we can go to whatever this Coalition thing is," says Anna.`

Reviewer 2: Minor writing suggestion, feel free to ignore.
```suggestion
			`	Her tone of voice becomes much more excited. "Do I have to pay anything? I can't afford off-world transport at all, let alone to a completely different segment of the galaxy! Actually, I know - I'll give you a copy of Beyond the Hyperdrive! Thank you so much! Just wait for a while. I'll go over to my house and bring what I'll need for the trip, and then we can go to whatever this Coalition thing is," says Anna.`
```

		conversation
			`As the flames of your jump drive fade away, Anna comes out of her bunk in shock. "What was that? It felt like the whole ship vanished out of our space altogether! Where are we?"`
			choice
				`	"My jump drive would like to welcome you to Coalition space."`
				`	"That was a jump drive."`
				`	"That's what happens every time I go to Coalition space with a jump drive."`
			`	"Wow. I can't believe this is happening!" Anna looks out the window, then turns back.`
			`	"Can I take a look at your jump drive? I'm interested to see how it works," Anna says. You take her to the engine room of <ship>, where she begins taking notes and pictures of the drive.`
			`	"Well, thank you for bringing me here," Anna says when she finishes. "I didn't sleep too well, so I'll see you later. Goodbye!" She heads back into her cabin, and you settle back into the controls.`
	npc
		government "Heliarch"
		personality heroic opportunistic
		fleet "Heliarch"
	on complete
		set "coalition: struggling author - beyond the hyperdrive"
		conversation
			`Anna gives you the book before you both exit the <ship>. The moment you are out, three Kimek approach you and Anna.`

> 			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"`

Reviewer 2: These civilians aren't at least a little shocked to see another human in Coalition space?

			`	You call Anna, and ask her for another copy of Beyond the Hyperdrive for a friend named Kachachee. She says, "Sure, I'll send it to you over the Coalition's incredible hyperspace relays. For something as small as a book, there probably won't even need to be a spaceship bringing it! Where do you want it sent?" You reply, "To <planet>." You should head there to receive and gift it to Kachachee.`
				accept
	on complete
		payment 7000
		conversation
			branch owncopy
				not "struggling author: asked for another copy of beyond the hyperdrive"
			`After receiving the book - it turns out to have gotten there in a normal spacecraft, along with a message from Anna stating her disappointment - you get in contact with Kachachee and inform them you have a copy of "Beyond the Hyperdrive." They respond, telling you to meet them at a public square within the spaceport.`
				goto gift
			label owncopy
			`When you land on <planet>, you get in contact with Kachachee and inform them you have a copy of "Beyond the Hyperdrive." They respond, telling you to meet them at a public square within the spaceport.`
			label gift

> 			`	Walking over to the aforementioned public square, you recognize Kachachee among the crowd. You hold up your copy of Beyond the Hyperdrive and hand it over to them. Kachachee eagerly takes the copy. "Very thankful we are, for this book. Read it with my group, I must. But for your troubles, worthy of <payment>, you are." They then give you a chip for <payment>.`

Reviewer 2: Reducing repetition. 
```suggestion
			`	Walking over to the aforementioned public square, you recognize Kachachee among the crowd. You hold up your copy of Beyond the Hyperdrive and hand it over to them. Kachachee eagerly takes the copy. "Very thankful we are, for this book. Read it with my group, I must. But for your troubles, worthy of <payment>, you are." They then give you a chip and walk away, blending back into the crowd.`
```

				goto questions
			label publishingcompany
			action
				clear "struggling author: arachi"
				clear "struggling author: saryd"
			`	"But where can we find a Kimek publishing company? We haven't even translated our book into their language yet!"`
			choice
				`	"We'll find a company."`
				`	"Given the number of books here, there has to be a publishing company somewhere."`
			`	"The faster we can get the book published, the better. Meet me in the spaceport in a few hours; I'll find a good company."`
			`	Before Anna can head off, Temporo rises. "Again accompany you, I will."`

mission "Coalition: Struggling Author 4"
	name "Kimek Publishing Company"
	description "Anna Fangvin found a Kimek publishing company who is willing to get her book published. Bring her to <destination>."
	source "Ki Patek Ka"
	destination "Inmost Blue"
	blocked "You do not have enough bunk space for Anna Fangvin and the Heliarch agent. Come back here when you have two bunks free."
	passengers 2
	to offer
		has "Coalition: Struggling Author 3: done"
	on offer
		conversation

> 			`The spaceport of Ki Patek Ka is vast, and would normally prove a nightmarish place to look for just one person. Thankfully, Anna being the only other human within the Coalition makes your search relatively quick. She has chosen to settle in a cafeteria with her laptop. She is typing away at the keyboard, presumably writing a short story to post.`

Reviewer 2: ```suggestion
			`The spaceport of Ki Patek Ka is vast, and would normally prove a nightmarish place to look for just one person. Thankfully, Anna being the only other human within the Coalition makes your search relatively quick. She has chosen to settle in a cafeteria with her laptop, typing away at the keyboard, presumably writing a short story to post.`
```

			action
				payment -40
				set "human cultural data"

> 			`You wondered how you would get a copy of humanity's entire cultural archive, but it turned out to be easier than expected. The station has a small museum gift shop, which sells a copy of the archives on a data card. ("The perfect gift for the budding historian in your family! Hold all of human history in the palm of your hand. On sale now for only 39.99!") Given that a cheap data card costs a fraction of a credit, it's a bit of a rip-off, but you gladly buy one to take back to Kachachee.`

Reviewer 3: Wait, we're really gonna diverge from the text that you get when you give the datachip to the Hai?

Author: Does it need to be the exact same?

Reviewer 3: It kinda messes with the consistency, so to speak.

Author: ```suggestion
			`You had wondered how you would get a copy of the entire cultural archive, but it turns out to be easier than expected. The station has a small museum gift shop, which sells a copy of the archives on a data card. ("The perfect gift for the budding historian in your family! Hold all of human history in the palm of your hand. On sale now for only 39.99!") Given that a cheap data card costs a fraction of a credit, it's a bit of a rip-off, but you gladly buy one to take back to Kachachee.`
```
